-- MS SQL version of testSQL.sql 
-- create new test database with test data inserted.

Use cbioportal_test_persistence;

IF OBJECT_ID('dbo.info', 'U') IS NOT NULL
 DROP TABLE dbo.info

IF OBJECT_ID('dbo.clinical_event_data', 'U') IS NOT NULL
 DROP TABLE dbo.clinical_event_data

IF OBJECT_ID('dbo.clinical_event', 'U') IS NOT NULL
 DROP TABLE dbo.clinical_event

IF OBJECT_ID('dbo.pdb_uniprot_residue_mapping', 'U') IS NOT NULL
 DROP TABLE dbo.pdb_uniprot_residue_mapping

IF OBJECT_ID('dbo.pdb_uniprot_alignment', 'U') IS NOT NULL
 DROP TABLE dbo.pdb_uniprot_alignment

IF OBJECT_ID('dbo.cosmic_mutation', 'U') IS NOT NULL
 DROP TABLE dbo.cosmic_mutation

IF OBJECT_ID('dbo.copy_number_seg_file', 'U') IS NOT NULL
 DROP TABLE dbo.copy_number_seg_file

IF OBJECT_ID('dbo.copy_number_seg', 'U') IS NOT NULL
 DROP TABLE dbo.copy_number_seg

IF OBJECT_ID('dbo.sample_cna_event', 'U') IS NOT NULL
 DROP TABLE dbo.sample_cna_event

IF OBJECT_ID('dbo.cna_event', 'U') IS NOT NULL
 DROP TABLE dbo.cna_event

IF OBJECT_ID('dbo.drug_interaction', 'U') IS NOT NULL
 DROP TABLE dbo.drug_interaction

IF OBJECT_ID('dbo.drug', 'U') IS NOT NULL
 DROP TABLE dbo.drug

IF OBJECT_ID('dbo.pfam_graphics', 'U') IS NOT NULL
 DROP TABLE dbo.pfam_graphics

IF OBJECT_ID('dbo.text_cache', 'U') IS NOT NULL
 DROP TABLE dbo.text_cache

IF OBJECT_ID('dbo.gistic_to_gene', 'U') IS NOT NULL
 DROP TABLE dbo.gistic_to_gene

IF OBJECT_ID('dbo.gistic', 'U') IS NOT NULL
 DROP TABLE dbo.gistic

IF OBJECT_ID('dbo.sanger_cancer_census', 'U') IS NOT NULL
 DROP TABLE dbo.sanger_cancer_census

IF OBJECT_ID('dbo.protein_array_cancer_study', 'U') IS NOT NULL
 DROP TABLE dbo.protein_array_cancer_study

IF OBJECT_ID('dbo.protein_array_data', 'U') IS NOT NULL
 DROP TABLE dbo.protein_array_data

IF OBJECT_ID('dbo.protein_array_target', 'U') IS NOT NULL
 DROP TABLE dbo.protein_array_target

IF OBJECT_ID('dbo.protein_array_info', 'U') IS NOT NULL
 DROP TABLE dbo.protein_array_info

IF OBJECT_ID('dbo.mut_sig', 'U') IS NOT NULL
 DROP TABLE dbo.mut_sig

IF OBJECT_ID('dbo.interaction', 'U') IS NOT NULL
 DROP TABLE dbo.interaction

IF OBJECT_ID('dbo.clinical_attribute_meta', 'U') IS NOT NULL
 DROP TABLE dbo.clinical_attribute_meta

IF OBJECT_ID('dbo.clinical_sample', 'U') IS NOT NULL
 DROP TABLE dbo.clinical_sample

IF OBJECT_ID('dbo.clinical_patient', 'U') IS NOT NULL
 DROP TABLE dbo.clinical_patient

IF OBJECT_ID('dbo.mutation_count', 'U') IS NOT NULL
 DROP TABLE dbo.mutation_count

IF OBJECT_ID('dbo.mutation', 'U') IS NOT NULL
 DROP TABLE dbo.mutation

IF OBJECT_ID('dbo.mutation_event', 'U') IS NOT NULL
 DROP TABLE dbo.mutation_event

IF OBJECT_ID('dbo.structural_variant', 'U') IS NOT NULL
 DROP TABLE dbo.structural_variant

IF OBJECT_ID('dbo.sample_profile', 'U') IS NOT NULL
 DROP TABLE dbo.sample_profile

IF OBJECT_ID('dbo.gene_panel_list', 'U') IS NOT NULL
 DROP TABLE dbo.gene_panel_list

IF OBJECT_ID('dbo.gene_panel', 'U') IS NOT NULL
 DROP TABLE dbo.gene_panel

IF OBJECT_ID('dbo.genetic_profile_samples', 'U') IS NOT NULL
 DROP TABLE dbo.genetic_profile_samples

IF OBJECT_ID('dbo.genetic_alteration', 'U') IS NOT NULL
 DROP TABLE dbo.genetic_alteration

IF OBJECT_ID('dbo.genetic_profile', 'U') IS NOT NULL
 DROP TABLE dbo.genetic_profile

IF OBJECT_ID('dbo.uniprot_id_mapping', 'U') IS NOT NULL
 DROP TABLE dbo.uniprot_id_mapping

IF OBJECT_ID('dbo.gene_alias', 'U') IS NOT NULL
 DROP TABLE dbo.gene_alias

IF OBJECT_ID('dbo.gene', 'U') IS NOT NULL
 DROP TABLE dbo.gene

IF OBJECT_ID('dbo.sample_list_list', 'U') IS NOT NULL
 DROP TABLE dbo.sample_list_list

IF OBJECT_ID('dbo.sample_list', 'U') IS NOT NULL
 DROP TABLE dbo.sample_list

IF OBJECT_ID('dbo.sample', 'U') IS NOT NULL
 DROP TABLE dbo.sample

IF OBJECT_ID('dbo.patient', 'U') IS NOT NULL
 DROP TABLE dbo.patient

IF OBJECT_ID('dbo.authorities', 'U') IS NOT NULL
 DROP TABLE dbo.authorities

IF OBJECT_ID('dbo.users', 'U') IS NOT NULL
 DROP TABLE dbo.users

IF OBJECT_ID('dbo.cancer_study', 'U') IS NOT NULL
 DROP TABLE dbo.cancer_study

IF OBJECT_ID('dbo.type_of_cancer', 'U') IS NOT NULL
 DROP TABLE dbo.type_of_cancer



GO

CREATE TABLE [type_of_cancer] (
   [TYPE_OF_CANCER_ID] VARCHAR(63) NOT NULL
,  [NAME] VARCHAR(255) NOT NULL
,  [CLINICAL_TRIAL_KEYWORDS] VARCHAR(1024) NOT NULL
,  [DEDICATED_COLOR] CHAR(31) NOT NULL
,  [SHORT_NAME] VARCHAR(127)
,  [PARENT] VARCHAR(63)
)

CREATE TABLE [cancer_study] (
   [CANCER_STUDY_ID] INTEGER NOT NULL IDENTITY(1,1)
,  [CANCER_STUDY_IDENTIFIER] VARCHAR(255)
,  [TYPE_OF_CANCER_ID] VARCHAR(63) NOT NULL
,  [NAME] VARCHAR(255) NOT NULL
,  [SHORT_NAME] VARCHAR(64) NOT NULL
,  [DESCRIPTION] VARCHAR(1024) NOT NULL
,  [PUBLIC] BIT NOT NULL
,  [PMID] VARCHAR(20) DEFAULT NULL
,  [CITATION] VARCHAR(200) DEFAULT NULL
,  [GROUPS] VARCHAR(200) DEFAULT NULL
,  [STATUS] INTEGER DEFAULT NULL
,  [IMPORT_DATE] DATETIME DEFAULT NULL
)

CREATE TABLE [users] (
   [EMAIL] VARCHAR(128) NOT NULL
,  [NAME] VARCHAR(255) NOT NULL
,  [ENABLED] BIT NOT NULL
)

CREATE TABLE [authorities] (
   [EMAIL] VARCHAR(128) NOT NULL
,  [AUTHORITY] VARCHAR(255) NOT NULL
)

CREATE TABLE [patient] (
   [INTERNAL_ID] INTEGER NOT NULL IDENTITY(1,1)
,  [STABLE_ID] VARCHAR(50) NOT NULL
,  [CANCER_STUDY_ID] INTEGER NOT NULL
)

CREATE TABLE [sample] (
   [INTERNAL_ID] INTEGER NOT NULL IDENTITY(1,1)
,  [STABLE_ID] VARCHAR(50) NOT NULL
,  [SAMPLE_TYPE] VARCHAR(255) NOT NULL
,  [PATIENT_ID] INTEGER NOT NULL
,  [TYPE_OF_CANCER_ID] VARCHAR(63) NOT NULL
)

CREATE TABLE [sample_list] (
   [LIST_ID] INTEGER NOT NULL IDENTITY(1,1)
,  [STABLE_ID] VARCHAR(255) NOT NULL
,  [CATEGORY] VARCHAR(255) NOT NULL
,  [CANCER_STUDY_ID] INTEGER NOT NULL
,  [NAME] VARCHAR(255) NOT NULL
,  [DESCRIPTION] TEXT
)

CREATE TABLE [sample_list_list] (
   [LIST_ID] INTEGER NOT NULL
,  [SAMPLE_ID] INTEGER NOT NULL
)

CREATE TABLE [gene] (
   [ENTREZ_GENE_ID] INTEGER NOT NULL
,  [HUGO_GENE_SYMBOL] VARCHAR(255) NOT NULL
,  [TYPE] VARCHAR(50)
,  [CYTOBAND] VARCHAR(50)
,  [LENGTH] INTEGER
)

CREATE TABLE [gene_alias] (
   [ENTREZ_GENE_ID] INTEGER NOT NULL
,  [GENE_ALIAS] VARCHAR(255) NOT NULL
)

CREATE TABLE [uniprot_id_mapping] (
   [UNIPROT_ACC] VARCHAR(255) NOT NULL
,  [UNIPROT_ID] VARCHAR(255) NOT NULL
,  [ENTREZ_GENE_ID] INTEGER NOT NULL
)

CREATE TABLE [genetic_profile] (
   [GENETIC_PROFILE_ID] INTEGER NOT NULL IDENTITY(1,1)
,  [STABLE_ID] VARCHAR(255) NOT NULL
,  [CANCER_STUDY_ID] INTEGER NOT NULL
,  [GENETIC_ALTERATION_TYPE] VARCHAR(255) NOT NULL
,  [DATATYPE] VARCHAR(255) NOT NULL
,  [NAME] VARCHAR(255) NOT NULL
,  [DESCRIPTION] TEXT
,  [SHOW_PROFILE_IN_ANALYSIS_TAB] BINARY(1) NOT NULL
)

CREATE TABLE [genetic_alteration] (
   [GENETIC_PROFILE_ID] INTEGER NOT NULL
,  [ENTREZ_GENE_ID] INTEGER NOT NULL
,  [VALUES] TEXT NOT NULL
)

CREATE TABLE [genetic_profile_samples] (
   [GENETIC_PROFILE_ID] INTEGER NOT NULL
,  [ORDERED_SAMPLE_LIST] TEXT NOT NULL
)

CREATE TABLE [gene_panel] (
   [INTERNAL_ID] INTEGER NOT NULL IDENTITY(1,1)
,  [STABLE_ID] VARCHAR(255) NOT NULL
,  [DESCRIPTION] TEXT
)

CREATE TABLE [gene_panel_list] (
   [INTERNAL_ID] INTEGER NOT NULL
,  [GENE_ID] INTEGER NOT NULL
)

CREATE TABLE [sample_profile] (
   [SAMPLE_ID] INTEGER NOT NULL
,  [GENETIC_PROFILE_ID] INTEGER NOT NULL
,  [PANEL_ID] INTEGER DEFAULT NULL
)

CREATE TABLE [structural_variant] (
   [SAMPLE_ID] INTEGER NOT NULL
,  [INTERNAL_ID] INTEGER NOT NULL IDENTITY(1,1)
,  [BREAKPOINT_TYPE] VARCHAR(25)
,  [ANNOTATION] VARCHAR(255)
,  [COMMENTS] VARCHAR(2048)
,  [CONFIDENCE_CLASS] VARCHAR(25)
,  [CONNECTION_TYPE] VARCHAR(25)
,  [EVENT_INFO] VARCHAR(255)
,  [MAPQ] INTEGER
,  [NORMAL_READ_COUNT] INTEGER
,  [NORMAL_VARIANT_COUNT] INTEGER
,  [PAIRED_END_READ_SUPPORT] VARCHAR(255)
,  [SITE1_CHROM] VARCHAR(25)
,  [SITE1_DESC] VARCHAR(255)
,  [SITE1_ENTREZ_GENE_ID] INTEGER
,  [SITE1_POS] INTEGER
,  [SITE2_CHROM] VARCHAR(25)
,  [SITE2_DESC] VARCHAR(255)
,  [SITE2_ENTREZ_GENE_ID] INTEGER
,  [SITE2_POS] INTEGER
,  [SPLIT_READ_SUPPORT] VARCHAR(255)
,  [SV_CLASS_NAME] VARCHAR(25)
,  [SV_DESC] VARCHAR(255)
,  [SV_LENGTH] INTEGER
,  [TUMOR_READ_COUNT] INTEGER
,  [TUMOR_VARIANT_COUNT] INTEGER
,  [VARIANT_STATUS_NAME] VARCHAR(255)
,  [GENETIC_PROFILE_ID] INTEGER NOT NULL
)

CREATE TABLE [mutation_event] (
   [MUTATION_EVENT_ID] INTEGER NOT NULL IDENTITY(1,1)
,  [ENTREZ_GENE_ID] INTEGER NOT NULL
,  [CHR] VARCHAR(5)
,  [START_POSITION] BIGINT
,  [END_POSITION] BIGINT
,  [REFERENCE_ALLELE] VARCHAR(400)
,  [TUMOR_SEQ_ALLELE] VARCHAR(400)
,  [PROTEIN_CHANGE] VARCHAR(255)
,  [MUTATION_TYPE] VARCHAR(255)
,  [FUNCTIONAL_IMPACT_SCORE] VARCHAR(50)
,  [FIS_VALUE] REAL
,  [LINK_XVAR] VARCHAR(500)
,  [LINK_PDB] VARCHAR(500)
,  [LINK_MSA] VARCHAR(500)
,  [NCBI_BUILD] VARCHAR(10)
,  [STRAND] VARCHAR(2)
,  [VARIANT_TYPE] VARCHAR(15)
,  [DB_SNP_RS] VARCHAR(25)
,  [DB_SNP_VAL_STATUS] VARCHAR(255)
,  [ONCOTATOR_DBSNP_RS] VARCHAR(255)
,  [ONCOTATOR_REFSEQ_MRNA_ID] VARCHAR(64)
,  [ONCOTATOR_CODON_CHANGE] VARCHAR(255)
,  [ONCOTATOR_UNIPROT_ENTRY_NAME] VARCHAR(64)
,  [ONCOTATOR_UNIPROT_ACCESSION] VARCHAR(64)
,  [ONCOTATOR_PROTEIN_POS_START] INTEGER
,  [ONCOTATOR_PROTEIN_POS_END] INTEGER
,  [CANONICAL_TRANSCRIPT] BIT
,  [KEYWORD] VARCHAR(50) DEFAULT NULL
)

CREATE TABLE [mutation] (
   [MUTATION_EVENT_ID] INTEGER NOT NULL
,  [GENETIC_PROFILE_ID] INTEGER NOT NULL
,  [SAMPLE_ID] INTEGER NOT NULL
,  [ENTREZ_GENE_ID] INTEGER NOT NULL
,  [CENTER] VARCHAR(100)
,  [SEQUENCER] VARCHAR(255)
,  [MUTATION_STATUS] VARCHAR(25)
,  [VALIDATION_STATUS] VARCHAR(25)
,  [TUMOR_SEQ_ALLELE1] VARCHAR(255)
,  [TUMOR_SEQ_ALLELE2] VARCHAR(255)
,  [MATCHED_NORM_SAMPLE_BARCODE] VARCHAR(255)
,  [MATCH_NORM_SEQ_ALLELE1] VARCHAR(255)
,  [MATCH_NORM_SEQ_ALLELE2] VARCHAR(255)
,  [TUMOR_VALIDATION_ALLELE1] VARCHAR(255)
,  [TUMOR_VALIDATION_ALLELE2] VARCHAR(255)
,  [MATCH_NORM_VALIDATION_ALLELE1] VARCHAR(255)
,  [MATCH_NORM_VALIDATION_ALLELE2] VARCHAR(255)
,  [VERIFICATION_STATUS] VARCHAR(10)
,  [SEQUENCING_PHASE] VARCHAR(100)
,  [SEQUENCE_SOURCE] VARCHAR(255) NOT NULL
,  [VALIDATION_METHOD] VARCHAR(255)
,  [SCORE] VARCHAR(100)
,  [BAM_FILE] VARCHAR(255)
,  [TUMOR_ALT_COUNT] INTEGER
,  [TUMOR_REF_COUNT] INTEGER
,  [NORMAL_ALT_COUNT] INTEGER
,  [NORMAL_REF_COUNT] INTEGER
,  [AMINO_ACID_CHANGE] VARCHAR(255)
)

CREATE TABLE [mutation_count] (
   [GENETIC_PROFILE_ID] INTEGER NOT NULL
,  [SAMPLE_ID] INTEGER NOT NULL
,  [MUTATION_COUNT] INTEGER NOT NULL
)

CREATE TABLE [clinical_patient] (
   [INTERNAL_ID] INTEGER NOT NULL
,  [ATTR_ID] VARCHAR(255) NOT NULL
,  [ATTR_VALUE] VARCHAR(255) NOT NULL
)

CREATE TABLE [clinical_sample] (
   [INTERNAL_ID] INTEGER NOT NULL
,  [ATTR_ID] VARCHAR(255) NOT NULL
,  [ATTR_VALUE] VARCHAR(255) NOT NULL
)

CREATE TABLE [clinical_attribute_meta] (
   [ATTR_ID] VARCHAR(255) NOT NULL
,  [DISPLAY_NAME] VARCHAR(255) NOT NULL
,  [DESCRIPTION] VARCHAR(2048) NOT NULL
,  [DATATYPE] VARCHAR(255) NOT NULL
,  [PATIENT_ATTRIBUTE] BIT NOT NULL
,  [PRIORITY] VARCHAR(255) NOT NULL
,  [CANCER_STUDY_ID] INTEGER NOT NULL
)

CREATE TABLE [interaction] (
   [GENE_A] BIGINT NOT NULL
,  [GENE_B] BIGINT NOT NULL
,  [INTERACTION_TYPE] VARCHAR(256) NOT NULL
,  [DATA_SOURCE] VARCHAR(256) NOT NULL
,  [EXPERIMENT_TYPES] VARCHAR(1024) NOT NULL
,  [PMIDS] VARCHAR(1024) NOT NULL
)

CREATE TABLE [mut_sig] (
   [CANCER_STUDY_ID] INTEGER NOT NULL
,  [ENTREZ_GENE_ID] INTEGER NOT NULL
,  [RANK] INTEGER NOT NULL
,  [NumBasesCovered] INTEGER NOT NULL
,  [NumMutations] INTEGER NOT NULL
,  [P_VALUE] REAL NOT NULL
,  [Q_VALUE] REAL NOT NULL
)

CREATE TABLE [protein_array_info] (
   [PROTEIN_ARRAY_ID] VARCHAR(50) NOT NULL
,  [TYPE] VARCHAR(50) NOT NULL
,  [GENE_SYMBOL] VARCHAR(50) NOT NULL
,  [TARGET_RESIDUE] VARCHAR(20) default NULL
)

CREATE TABLE [protein_array_target] (
   [PROTEIN_ARRAY_ID] VARCHAR(50) NOT NULL
,  [ENTREZ_GENE_ID] INTEGER NOT NULL
)

CREATE TABLE [protein_array_data] (
   [PROTEIN_ARRAY_ID] VARCHAR(50) NOT NULL
,  [CANCER_STUDY_ID] INTEGER NOT NULL
,  [SAMPLE_ID] INTEGER NOT NULL
,  [ABUNDANCE] FLOAT NOT NULL
)

CREATE TABLE [protein_array_cancer_study] (
   [PROTEIN_ARRAY_ID] VARCHAR(50) NOT NULL
,  [CANCER_STUDY_ID] INTEGER NOT NULL
)

CREATE TABLE [sanger_cancer_census] (
   [ENTREZ_GENE_ID] INTEGER NOT NULL
,  [CANCER_SOMATIC_MUT] TINYINT NOT NULL
,  [CANCER_GERMLINE_MUT] TINYINT NOT NULL
,  [TUMOR_TYPES_SOMATIC_MUT] TEXT NOT NULL
,  [TUMOR_TYPES_GERMLINE_MUT] TEXT NOT NULL
,  [CANCER_SYNDROME] TEXT NOT NULL
,  [TISSUE_TYPE] TEXT NOT NULL
,  [MUTATION_TYPE] TEXT NOT NULL
,  [TRANSLOCATION_PARTNER] TEXT NOT NULL
,  [OTHER_GERMLINE_MUT] TINYINT NOT NULL
,  [OTHER_DISEASE] TEXT NOT NULL
)

CREATE TABLE [gistic] (
   [GISTIC_ROI_ID] BIGINT NOT NULL IDENTITY(1,1)
,  [CANCER_STUDY_ID] INTEGER NOT NULL
,  [CHROMOSOME] INTEGER NOT NULL
,  [CYTOBAND] VARCHAR(255) NOT NULL
,  [WIDE_PEAK_START] INTEGER NOT NULL
,  [WIDE_PEAK_END] INTEGER NOT NULL
,  [Q_VALUE] FLOAT NOT NULL
,  [AMP] TINYINT NOT NULL
)

CREATE TABLE [gistic_to_gene] (
   [GISTIC_ROI_ID] BIGINT NOT NULL
,  [ENTREZ_GENE_ID] INTEGER NOT NULL
)

CREATE TABLE [text_cache] (
   [HASH_KEY] VARCHAR(32) NOT NULL
,  [TEXT] TEXT NOT NULL
,  [DATE_TIME_STAMP] DATETIME NOT NULL
)

CREATE TABLE [pfam_graphics] (
   [UNIPROT_ACC] VARCHAR(255) NOT NULL
,  [JSON_DATA] TEXT NOT NULL
)

CREATE TABLE [drug] (
   [DRUG_ID] CHAR(30) NOT NULL
,  [DRUG_RESOURCE] VARCHAR(255) NOT NULL
,  [DRUG_NAME] VARCHAR(255) NOT NULL
,  [DRUG_SYNONYMS] VARCHAR(4096) DEFAULT NULL
,  [DRUG_DESCRIPTION] VARCHAR(4096) DEFAULT NULL
,  [DRUG_XREF] VARCHAR(4096) DEFAULT NULL
,  [DRUG_ATC_CODE] VARCHAR(1024) DEFAULT NULL
,  [DRUG_APPROVED] INTEGER DEFAULT 0
,  [DRUG_CANCERDRUG] INTEGER DEFAULT 0
,  [DRUG_NUTRACEUTICAL] INTEGER DEFAULT 0
,  [DRUG_NUMOFTRIALS] INTEGER DEFAULT -1
)

CREATE TABLE [drug_interaction] (
   [DRUG] CHAR(30) NOT NULL
,  [TARGET] BIGINT NOT NULL
,  [INTERACTION_TYPE] CHAR(50) NOT NULL
,  [DATA_SOURCE] VARCHAR(256) NOT NULL
,  [EXPERIMENT_TYPES] VARCHAR(1024) DEFAULT NULL
,  [PMIDS] VARCHAR(1024) DEFAULT NULL
)

CREATE TABLE [cna_event] (
   [CNA_EVENT_ID] INTEGER NOT NULL IDENTITY(1,1)
,  [ENTREZ_GENE_ID] INTEGER NOT NULL
,  [ALTERATION] SMALLINT NOT NULL
)

CREATE TABLE [sample_cna_event] (
   [CNA_EVENT_ID] INTEGER NOT NULL
,  [SAMPLE_ID] INTEGER NOT NULL
,  [GENETIC_PROFILE_ID] INTEGER NOT NULL
)

CREATE TABLE [copy_number_seg] (
   [SEG_ID] INTEGER NOT NULL IDENTITY(1,1)
,  [CANCER_STUDY_ID] INTEGER NOT NULL
,  [SAMPLE_ID] INTEGER NOT NULL
,  [CHR] VARCHAR(5) NOT NULL
,  [START] INTEGER NOT NULL
,  [END] INTEGER NOT NULL
,  [NUM_PROBES] INTEGER NOT NULL
,  [SEGMENT_MEAN] FLOAT NOT NULL
)

CREATE TABLE [copy_number_seg_file] (
   [SEG_FILE_ID] INTEGER NOT NULL IDENTITY(1,1)
,  [CANCER_STUDY_ID] INTEGER NOT NULL
,  [REFERENCE_GENOME_ID] VARCHAR(10) NOT NULL
,  [DESCRIPTION] VARCHAR(255) NOT NULL
,  [FILENAME] VARCHAR(255) NOT NULL
)

CREATE TABLE [cosmic_mutation] (
   [COSMIC_MUTATION_ID] VARCHAR(30) NOT NULL
,  [CHR] VARCHAR(5)
,  [START_POSITION] BIGINT
,  [REFERENCE_ALLELE] VARCHAR(255)
,  [TUMOR_SEQ_ALLELE] VARCHAR(255)
,  [STRAND] VARCHAR(2)
,  [CODON_CHANGE] VARCHAR(255)
,  [ENTREZ_GENE_ID] INTEGER NOT NULL
,  [PROTEIN_CHANGE] VARCHAR(255) NOT NULL
,  [COUNT] INTEGER NOT NULL
,  [KEYWORD] VARCHAR(50) DEFAULT NULL
)

CREATE TABLE [pdb_uniprot_alignment] (
   [ALIGNMENT_ID] INTEGER NOT NULL
,  [PDB_ID] CHAR(4) NOT NULL
,  [CHAIN] CHAR(1) NOT NULL
,  [UNIPROT_ID] VARCHAR(50) NOT NULL
,  [PDB_FROM] VARCHAR(10) NOT NULL
,  [PDB_TO] VARCHAR(10) NOT NULL
,  [UNIPROT_FROM] INTEGER NOT NULL
,  [UNIPROT_TO] INTEGER NOT NULL
,  [EVALUE] REAL
,  [IDENTITY] REAL
,  [IDENTP] REAL
,  [UNIPROT_ALIGN] TEXT
,  [PDB_ALIGN] TEXT
,  [MIDLINE_ALIGN] TEXT
)

CREATE TABLE [pdb_uniprot_residue_mapping] (
   [ALIGNMENT_ID] INTEGER NOT NULL
,  [PDB_POSITION] INTEGER NOT NULL
,  [PDB_INSERTION_CODE] CHAR(1) DEFAULT NULL
,  [UNIPROT_POSITION] INTEGER NOT NULL
,  [MATCH] CHAR(1)
)

CREATE TABLE [clinical_event] (
   [CLINICAL_EVENT_ID] INTEGER NOT NULL IDENTITY(1,1)
,  [PATIENT_ID] INTEGER NOT NULL
,  [START_DATE] INTEGER NOT NULL
,  [STOP_DATE] INTEGER
,  [EVENT_TYPE] VARCHAR(20) NOT NULL
)

CREATE TABLE [clinical_event_data] (
   [CLINICAL_EVENT_ID] INTEGER NOT NULL
,  [KEY] VARCHAR(255) NOT NULL
,  [VALUE] VARCHAR(5000) NOT NULL
)

CREATE TABLE [info] (
   [DB_SCHEMA_VERSION] VARCHAR(24)
)



GO

-- MS SQL key and index definitions ported from MySQL version at C:\workspaces\git\sciserver-cbioportal\core\src\main\resources\db\cgds_fixed4MSSQL.sql

ALTER TABLE [type_of_cancer] ADD CONSTRAINT pk_type_of_cancer PRIMARY KEY ([TYPE_OF_CANCER_ID])

ALTER TABLE [cancer_study] ADD CONSTRAINT pk_cancer_study PRIMARY KEY ([CANCER_STUDY_ID])


-- TODO DEFINE: FOREIGNKEY (`TYPE_OF_CANCER_ID`) REFERENCES `type_of_cancer` (`TYPE_OF_CANCER_ID`)
ALTER TABLE cancer_study ADD CONSTRAINT fk_cancer_study_TYPE_OF_CANCER_ID FOREIGN KEY(TYPE_OF_CANCER_ID) REFERENCES [type_of_cancer](TYPE_OF_CANCER_ID)

ALTER TABLE [users] ADD CONSTRAINT pk_users PRIMARY KEY ([EMAIL])

ALTER TABLE [patient] ADD CONSTRAINT pk_patient PRIMARY KEY ([INTERNAL_ID])


-- TODO DEFINE: FOREIGNKEY (`CANCER_STUDY_ID`) REFERENCES `cancer_study` (`CANCER_STUDY_ID`) ON DELETE CASCADE
ALTER TABLE patient ADD CONSTRAINT fk_patient_CANCER_STUDY_ID FOREIGN KEY(CANCER_STUDY_ID) REFERENCES [cancer_study](CANCER_STUDY_ID)

ALTER TABLE [sample] ADD CONSTRAINT pk_sample PRIMARY KEY ([INTERNAL_ID])


-- TODO DEFINE: FOREIGNKEY (`PATIENT_ID`) REFERENCES `patient` (`INTERNAL_ID`) ON DELETE CASCADE
ALTER TABLE sample ADD CONSTRAINT fk_sample_PATIENT_ID FOREIGN KEY(PATIENT_ID) REFERENCES [patient](INTERNAL_ID)


-- TODO DEFINE: FOREIGNKEY (`TYPE_OF_CANCER_ID`) REFERENCES `type_of_cancer` (`TYPE_OF_CANCER_ID`)
ALTER TABLE sample ADD CONSTRAINT fk_sample_TYPE_OF_CANCER_ID FOREIGN KEY(TYPE_OF_CANCER_ID) REFERENCES [type_of_cancer](TYPE_OF_CANCER_ID)

ALTER TABLE [sample_list] ADD CONSTRAINT pk_sample_list PRIMARY KEY ([LIST_ID])


-- TODO DEFINE: FOREIGNKEY (`CANCER_STUDY_ID`) REFERENCES `cancer_study` (`CANCER_STUDY_ID`) ON DELETE CASCADE
ALTER TABLE sample_list ADD CONSTRAINT fk_sample_list_CANCER_STUDY_ID FOREIGN KEY(CANCER_STUDY_ID) REFERENCES [cancer_study](CANCER_STUDY_ID)

ALTER TABLE [sample_list_list] ADD CONSTRAINT pk_sample_list_list PRIMARY KEY ([LIST_ID],[SAMPLE_ID])


-- TODO DEFINE: FOREIGNKEY (`SAMPLE_ID`) REFERENCES `sample` (`INTERNAL_ID`) ON DELETE CASCADE
ALTER TABLE sample_list_list ADD CONSTRAINT fk_sample_list_list_SAMPLE_ID FOREIGN KEY(SAMPLE_ID) REFERENCES [sample](INTERNAL_ID)

ALTER TABLE [gene] ADD CONSTRAINT pk_gene PRIMARY KEY ([ENTREZ_GENE_ID])



 -- TODO DEFINE: SINGLEKEY `HUGO_GENE_SYMBOL` (`HUGO_GENE_SYMBOL`)
CREATE INDEX ix_gene_HUGO_GENE_SYMBOL ON gene([HUGO_GENE_SYMBOL])
ALTER TABLE [gene_alias] ADD CONSTRAINT pk_gene_alias PRIMARY KEY ([ENTREZ_GENE_ID],[GENE_ALIAS])


-- TODO DEFINE: FOREIGNKEY (`ENTREZ_GENE_ID`) REFERENCES `gene` (`ENTREZ_GENE_ID`)
ALTER TABLE gene_alias ADD CONSTRAINT fk_gene_alias_ENTREZ_GENE_ID FOREIGN KEY(ENTREZ_GENE_ID) REFERENCES [gene](ENTREZ_GENE_ID)

ALTER TABLE [uniprot_id_mapping] ADD CONSTRAINT pk_uniprot_id_mapping PRIMARY KEY ([ENTREZ_GENE_ID],[UNIPROT_ID])



 -- TODO DEFINE: SINGLEKEY (`UNIPROT_ID`)
CREATE INDEX ix_uniprot_id_mapping_UNIPROT_ID ON uniprot_id_mapping([UNIPROT_ID])


 -- TODO DEFINE: SINGLEKEY (`UNIPROT_ACC`)
CREATE INDEX ix_uniprot_id_mapping_UNIPROT_ACC ON uniprot_id_mapping([UNIPROT_ACC])

-- TODO DEFINE: FOREIGNKEY (`ENTREZ_GENE_ID`) REFERENCES `gene` (`ENTREZ_GENE_ID`)
ALTER TABLE uniprot_id_mapping ADD CONSTRAINT fk_uniprot_id_mapping_ENTREZ_GENE_ID FOREIGN KEY(ENTREZ_GENE_ID) REFERENCES [gene](ENTREZ_GENE_ID)

ALTER TABLE [genetic_profile] ADD CONSTRAINT pk_genetic_profile PRIMARY KEY ([GENETIC_PROFILE_ID])


-- TODO DEFINE: FOREIGNKEY (`CANCER_STUDY_ID`) REFERENCES `cancer_study` (`CANCER_STUDY_ID`) ON DELETE CASCADE
ALTER TABLE genetic_profile ADD CONSTRAINT fk_genetic_profile_CANCER_STUDY_ID FOREIGN KEY(CANCER_STUDY_ID) REFERENCES [cancer_study](CANCER_STUDY_ID)



 -- TODO DEFINE: SINGLEKEY `QUICK_LOOK_UP` (`ENTREZ_GENE_ID`)
CREATE INDEX ix_genetic_alteration_ENTREZ_GENE_ID ON genetic_alteration([ENTREZ_GENE_ID])


 -- TODO DEFINE: SINGLEKEY `QUICK_LOOK_UP2` (`ENTREZ_GENE_ID`,`GENETIC_PROFILE_ID`)
CREATE INDEX ix_genetic_alteration_ENTREZ_GENE_ID_GENETIC_PROFILE_ID ON genetic_alteration([ENTREZ_GENE_ID],[GENETIC_PROFILE_ID])

-- TODO DEFINE: FOREIGNKEY (`ENTREZ_GENE_ID`) REFERENCES `gene` (`ENTREZ_GENE_ID`)
ALTER TABLE genetic_alteration ADD CONSTRAINT fk_genetic_alteration_ENTREZ_GENE_ID FOREIGN KEY(ENTREZ_GENE_ID) REFERENCES [gene](ENTREZ_GENE_ID)


-- TODO DEFINE: FOREIGNKEY (`GENETIC_PROFILE_ID`) REFERENCES `genetic_profile` (`GENETIC_PROFILE_ID`) ON DELETE CASCADE
ALTER TABLE genetic_alteration ADD CONSTRAINT fk_genetic_alteration_GENETIC_PROFILE_ID FOREIGN KEY(GENETIC_PROFILE_ID) REFERENCES [genetic_profile](GENETIC_PROFILE_ID)


-- TODO DEFINE: FOREIGNKEY (`GENETIC_PROFILE_ID`) REFERENCES `genetic_profile` (`GENETIC_PROFILE_ID`) ON DELETE CASCADE
ALTER TABLE genetic_profile_samples ADD CONSTRAINT fk_genetic_profile_samples_GENETIC_PROFILE_ID FOREIGN KEY(GENETIC_PROFILE_ID) REFERENCES [genetic_profile](GENETIC_PROFILE_ID)

ALTER TABLE [gene_panel] ADD CONSTRAINT pk_gene_panel PRIMARY KEY ([INTERNAL_ID])

ALTER TABLE [gene_panel_list] ADD CONSTRAINT pk_gene_panel_list PRIMARY KEY ([INTERNAL_ID],[GENE_ID])


-- TODO DEFINE: FOREIGNKEY (`INTERNAL_ID`) REFERENCES `gene_panel` (`INTERNAL_ID`) ON DELETE CASCADE
ALTER TABLE gene_panel_list ADD CONSTRAINT fk_gene_panel_list_INTERNAL_ID FOREIGN KEY(INTERNAL_ID) REFERENCES [gene_panel](INTERNAL_ID)


-- TODO DEFINE: FOREIGNKEY (`GENE_ID`) REFERENCES `gene` (`ENTREZ_GENE_ID`) ON DELETE CASCADE
ALTER TABLE gene_panel_list ADD CONSTRAINT fk_gene_panel_list_GENE_ID FOREIGN KEY(GENE_ID) REFERENCES [gene](ENTREZ_GENE_ID)



 CREATE INDEX ix_sample_profile_UQ_SAMPLE_ID_GENETIC_PROFILE_ID ON [sample_profile]([SAMPLE_ID],[GENETIC_PROFILE_ID])

 -- TODO DEFINE: SINGLEKEY (`SAMPLE_ID`)
CREATE INDEX ix_sample_profile_SAMPLE_ID ON sample_profile([SAMPLE_ID])

-- TODO DEFINE: FOREIGNKEY (`GENETIC_PROFILE_ID`) REFERENCES `genetic_profile` (`GENETIC_PROFILE_ID`) ON DELETE CASCADE
ALTER TABLE sample_profile ADD CONSTRAINT fk_sample_profile_GENETIC_PROFILE_ID FOREIGN KEY(GENETIC_PROFILE_ID) REFERENCES [genetic_profile](GENETIC_PROFILE_ID)


-- TODO DEFINE: FOREIGNKEY (`SAMPLE_ID`) REFERENCES `sample` (`INTERNAL_ID`) ON DELETE CASCADE
ALTER TABLE sample_profile ADD CONSTRAINT fk_sample_profile_SAMPLE_ID FOREIGN KEY(SAMPLE_ID) REFERENCES [sample](INTERNAL_ID)


-- TODO DEFINE: FOREIGNKEY (`PANEL_ID`) REFERENCES `gene_panel` (`INTERNAL_ID`) ON DELETE RESTRICT
ALTER TABLE sample_profile ADD CONSTRAINT fk_sample_profile_PANEL_ID FOREIGN KEY(PANEL_ID) REFERENCES [gene_panel](INTERNAL_ID)

ALTER TABLE [structural_variant] ADD CONSTRAINT pk_structural_variant PRIMARY KEY ([INTERNAL_ID])


-- TODO DEFINE: FOREIGNKEY (`SAMPLE_ID`) REFERENCES `sample` (`INTERNAL_ID`) ON DELETE CASCADE
ALTER TABLE structural_variant ADD CONSTRAINT fk_structural_variant_SAMPLE_ID FOREIGN KEY(SAMPLE_ID) REFERENCES [sample](INTERNAL_ID)


-- TODO DEFINE: FOREIGNKEY (`SITE1_ENTREZ_GENE_ID`) REFERENCES `gene` (`ENTREZ_GENE_ID`) ON DELETE CASCADE
ALTER TABLE structural_variant ADD CONSTRAINT fk_structural_variant_SITE1_ENTREZ_GENE_ID FOREIGN KEY(SITE1_ENTREZ_GENE_ID) REFERENCES [gene](ENTREZ_GENE_ID)


-- TODO DEFINE: FOREIGNKEY (`SITE2_ENTREZ_GENE_ID`) REFERENCES `gene` (`ENTREZ_GENE_ID`) ON DELETE CASCADE
ALTER TABLE structural_variant ADD CONSTRAINT fk_structural_variant_SITE2_ENTREZ_GENE_ID FOREIGN KEY(SITE2_ENTREZ_GENE_ID) REFERENCES [gene](ENTREZ_GENE_ID)


-- TODO DEFINE: FOREIGNKEY (`GENETIC_PROFILE_ID`) REFERENCES `genetic_profile` (`GENETIC_PROFILE_ID`) ON DELETE CASCADE
ALTER TABLE structural_variant ADD CONSTRAINT fk_structural_variant_GENETIC_PROFILE_ID FOREIGN KEY(GENETIC_PROFILE_ID) REFERENCES [genetic_profile](GENETIC_PROFILE_ID)

ALTER TABLE [mutation_event] ADD CONSTRAINT pk_mutation_event PRIMARY KEY ([MUTATION_EVENT_ID])



 -- TODO DEFINE: SINGLEKEY (`KEYWORD`)
CREATE INDEX ix_mutation_event_KEYWORD ON mutation_event([KEYWORD])

-- TODO DEFINE: FOREIGNKEY (`ENTREZ_GENE_ID`) REFERENCES `gene` (`ENTREZ_GENE_ID`)
ALTER TABLE mutation_event ADD CONSTRAINT fk_mutation_event_ENTREZ_GENE_ID FOREIGN KEY(ENTREZ_GENE_ID) REFERENCES [gene](ENTREZ_GENE_ID)



 CREATE INDEX ix_mutation_UQ_MUTATION_EVENT_ID_GENETIC_PROFILE_ID_SAMPLE_ID ON [mutation]([MUTATION_EVENT_ID],[GENETIC_PROFILE_ID],[SAMPLE_ID])

 -- TODO DEFINE: SINGLEKEY (`GENETIC_PROFILE_ID`,`ENTREZ_GENE_ID`)
CREATE INDEX ix_mutation_GENETIC_PROFILE_ID_ENTREZ_GENE_ID ON mutation([GENETIC_PROFILE_ID],[ENTREZ_GENE_ID])


 -- TODO DEFINE: SINGLEKEY (`GENETIC_PROFILE_ID`,`SAMPLE_ID`)
CREATE INDEX ix_mutation_GENETIC_PROFILE_ID_SAMPLE_ID ON mutation([GENETIC_PROFILE_ID],[SAMPLE_ID])


 -- TODO DEFINE: SINGLEKEY (`GENETIC_PROFILE_ID`)
CREATE INDEX ix_mutation_GENETIC_PROFILE_ID ON mutation([GENETIC_PROFILE_ID])


 -- TODO DEFINE: SINGLEKEY (`ENTREZ_GENE_ID`)
CREATE INDEX ix_mutation_ENTREZ_GENE_ID ON mutation([ENTREZ_GENE_ID])


 -- TODO DEFINE: SINGLEKEY (`SAMPLE_ID`)
CREATE INDEX ix_mutation_SAMPLE_ID ON mutation([SAMPLE_ID])


 -- TODO DEFINE: SINGLEKEY (`MUTATION_EVENT_ID`)
CREATE INDEX ix_mutation_MUTATION_EVENT_ID ON mutation([MUTATION_EVENT_ID])

-- TODO DEFINE: FOREIGNKEY (`MUTATION_EVENT_ID`) REFERENCES `mutation_event` (`MUTATION_EVENT_ID`)
ALTER TABLE mutation ADD CONSTRAINT fk_mutation_MUTATION_EVENT_ID FOREIGN KEY(MUTATION_EVENT_ID) REFERENCES [mutation_event](MUTATION_EVENT_ID)


-- TODO DEFINE: FOREIGNKEY (`ENTREZ_GENE_ID`) REFERENCES `gene` (`ENTREZ_GENE_ID`)
ALTER TABLE mutation ADD CONSTRAINT fk_mutation_ENTREZ_GENE_ID FOREIGN KEY(ENTREZ_GENE_ID) REFERENCES [gene](ENTREZ_GENE_ID)


-- TODO DEFINE: FOREIGNKEY (`GENETIC_PROFILE_ID`) REFERENCES `genetic_profile` (`GENETIC_PROFILE_ID`) ON DELETE CASCADE
ALTER TABLE mutation ADD CONSTRAINT fk_mutation_GENETIC_PROFILE_ID FOREIGN KEY(GENETIC_PROFILE_ID) REFERENCES [genetic_profile](GENETIC_PROFILE_ID)


-- TODO DEFINE: FOREIGNKEY (`SAMPLE_ID`) REFERENCES `sample` (`INTERNAL_ID`) ON DELETE CASCADE
ALTER TABLE mutation ADD CONSTRAINT fk_mutation_SAMPLE_ID FOREIGN KEY(SAMPLE_ID) REFERENCES [sample](INTERNAL_ID)



 -- TODO DEFINE: SINGLEKEY (`GENETIC_PROFILE_ID`,`SAMPLE_ID`)
CREATE INDEX ix_mutation_count_GENETIC_PROFILE_ID_SAMPLE_ID ON mutation_count([GENETIC_PROFILE_ID],[SAMPLE_ID])

-- TODO DEFINE: FOREIGNKEY (`GENETIC_PROFILE_ID`) REFERENCES `genetic_profile` (`GENETIC_PROFILE_ID`) ON DELETE CASCADE
ALTER TABLE mutation_count ADD CONSTRAINT fk_mutation_count_GENETIC_PROFILE_ID FOREIGN KEY(GENETIC_PROFILE_ID) REFERENCES [genetic_profile](GENETIC_PROFILE_ID)


-- TODO DEFINE: FOREIGNKEY (`SAMPLE_ID`) REFERENCES `sample` (`INTERNAL_ID`) ON DELETE CASCADE
ALTER TABLE mutation_count ADD CONSTRAINT fk_mutation_count_SAMPLE_ID FOREIGN KEY(SAMPLE_ID) REFERENCES [sample](INTERNAL_ID)

ALTER TABLE [clinical_patient] ADD CONSTRAINT pk_clinical_patient PRIMARY KEY ([INTERNAL_ID],[ATTR_ID])


-- TODO DEFINE: FOREIGNKEY (`INTERNAL_ID`) REFERENCES `patient` (`INTERNAL_ID`) ON DELETE CASCADE
ALTER TABLE clinical_patient ADD CONSTRAINT fk_clinical_patient_INTERNAL_ID FOREIGN KEY(INTERNAL_ID) REFERENCES [patient](INTERNAL_ID)

ALTER TABLE [clinical_sample] ADD CONSTRAINT pk_clinical_sample PRIMARY KEY ([INTERNAL_ID],[ATTR_ID])


-- TODO DEFINE: FOREIGNKEY (`INTERNAL_ID`) REFERENCES `sample` (`INTERNAL_ID`) ON DELETE CASCADE
ALTER TABLE clinical_sample ADD CONSTRAINT fk_clinical_sample_INTERNAL_ID FOREIGN KEY(INTERNAL_ID) REFERENCES [sample](INTERNAL_ID)

ALTER TABLE [clinical_attribute_meta] ADD CONSTRAINT pk_clinical_attribute_meta PRIMARY KEY ([ATTR_ID],[CANCER_STUDY_ID])


-- TODO DEFINE: FOREIGNKEY (`CANCER_STUDY_ID`) REFERENCES `cancer_study` (`CANCER_STUDY_ID`) ON DELETE CASCADE
ALTER TABLE clinical_attribute_meta ADD CONSTRAINT fk_clinical_attribute_meta_CANCER_STUDY_ID FOREIGN KEY(CANCER_STUDY_ID) REFERENCES [cancer_study](CANCER_STUDY_ID)

ALTER TABLE [mut_sig] ADD CONSTRAINT pk_mut_sig PRIMARY KEY ([CANCER_STUDY_ID],[ENTREZ_GENE_ID])


-- TODO DEFINE: FOREIGNKEY (`CANCER_STUDY_ID`) REFERENCES `cancer_study` (`CANCER_STUDY_ID`) ON DELETE CASCADE
ALTER TABLE mut_sig ADD CONSTRAINT fk_mut_sig_CANCER_STUDY_ID FOREIGN KEY(CANCER_STUDY_ID) REFERENCES [cancer_study](CANCER_STUDY_ID)


-- TODO DEFINE: FOREIGNKEY (`ENTREZ_GENE_ID`) REFERENCES `gene` (`ENTREZ_GENE_ID`)
ALTER TABLE mut_sig ADD CONSTRAINT fk_mut_sig_ENTREZ_GENE_ID FOREIGN KEY(ENTREZ_GENE_ID) REFERENCES [gene](ENTREZ_GENE_ID)

ALTER TABLE [protein_array_info] ADD CONSTRAINT pk_protein_array_info PRIMARY KEY ([PROTEIN_ARRAY_ID])

ALTER TABLE [protein_array_target] ADD CONSTRAINT pk_protein_array_target PRIMARY KEY ([PROTEIN_ARRAY_ID],[ENTREZ_GENE_ID])


-- TODO DEFINE: FOREIGNKEY (`ENTREZ_GENE_ID`) REFERENCES `gene` (`ENTREZ_GENE_ID`)
ALTER TABLE protein_array_target ADD CONSTRAINT fk_protein_array_target_ENTREZ_GENE_ID FOREIGN KEY(ENTREZ_GENE_ID) REFERENCES [gene](ENTREZ_GENE_ID)


-- TODO DEFINE: FOREIGNKEY (`PROTEIN_ARRAY_ID`) REFERENCES `protein_array_info` (`PROTEIN_ARRAY_ID`)
ALTER TABLE protein_array_target ADD CONSTRAINT fk_protein_array_target_PROTEIN_ARRAY_ID FOREIGN KEY(PROTEIN_ARRAY_ID) REFERENCES [protein_array_info](PROTEIN_ARRAY_ID)

ALTER TABLE [protein_array_data] ADD CONSTRAINT pk_protein_array_data PRIMARY KEY ([PROTEIN_ARRAY_ID],[CANCER_STUDY_ID],[SAMPLE_ID])


-- TODO DEFINE: FOREIGNKEY (`PROTEIN_ARRAY_ID`) REFERENCES `protein_array_info` (`PROTEIN_ARRAY_ID`)
ALTER TABLE protein_array_data ADD CONSTRAINT fk_protein_array_data_PROTEIN_ARRAY_ID FOREIGN KEY(PROTEIN_ARRAY_ID) REFERENCES [protein_array_info](PROTEIN_ARRAY_ID)


-- TODO DEFINE: FOREIGNKEY (`CANCER_STUDY_ID`) REFERENCES `cancer_study` (`CANCER_STUDY_ID`) ON DELETE CASCADE
ALTER TABLE protein_array_data ADD CONSTRAINT fk_protein_array_data_CANCER_STUDY_ID FOREIGN KEY(CANCER_STUDY_ID) REFERENCES [cancer_study](CANCER_STUDY_ID)


-- TODO DEFINE: FOREIGNKEY (`SAMPLE_ID`) REFERENCES `sample` (`INTERNAL_ID`) ON DELETE CASCADE
ALTER TABLE protein_array_data ADD CONSTRAINT fk_protein_array_data_SAMPLE_ID FOREIGN KEY(SAMPLE_ID) REFERENCES [sample](INTERNAL_ID)

ALTER TABLE [protein_array_cancer_study] ADD CONSTRAINT pk_protein_array_cancer_study PRIMARY KEY ([PROTEIN_ARRAY_ID],[CANCER_STUDY_ID])


-- TODO DEFINE: FOREIGNKEY (`CANCER_STUDY_ID`) REFERENCES `cancer_study` (`CANCER_STUDY_ID`) ON DELETE CASCADE
ALTER TABLE protein_array_cancer_study ADD CONSTRAINT fk_protein_array_cancer_study_CANCER_STUDY_ID FOREIGN KEY(CANCER_STUDY_ID) REFERENCES [cancer_study](CANCER_STUDY_ID)


-- TODO DEFINE: FOREIGNKEY (`ENTREZ_GENE_ID`) REFERENCES `gene` (`ENTREZ_GENE_ID`)
ALTER TABLE sanger_cancer_census ADD CONSTRAINT fk_sanger_cancer_census_ENTREZ_GENE_ID FOREIGN KEY(ENTREZ_GENE_ID) REFERENCES [gene](ENTREZ_GENE_ID)

ALTER TABLE [gistic] ADD CONSTRAINT pk_gistic PRIMARY KEY ([GISTIC_ROI_ID])


-- TODO DEFINE: FOREIGNKEY (`CANCER_STUDY_ID`) REFERENCES `cancer_study` (`CANCER_STUDY_ID`) ON DELETE CASCADE
ALTER TABLE gistic ADD CONSTRAINT fk_gistic_CANCER_STUDY_ID FOREIGN KEY(CANCER_STUDY_ID) REFERENCES [cancer_study](CANCER_STUDY_ID)

ALTER TABLE [gistic_to_gene] ADD CONSTRAINT pk_gistic_to_gene PRIMARY KEY ([GISTIC_ROI_ID],[ENTREZ_GENE_ID])


-- TODO DEFINE: FOREIGNKEY (`ENTREZ_GENE_ID`) REFERENCES `gene` (`ENTREZ_GENE_ID`)
ALTER TABLE gistic_to_gene ADD CONSTRAINT fk_gistic_to_gene_ENTREZ_GENE_ID FOREIGN KEY(ENTREZ_GENE_ID) REFERENCES [gene](ENTREZ_GENE_ID)


-- TODO DEFINE: FOREIGNKEY (`GISTIC_ROI_ID`) REFERENCES `gistic` (`GISTIC_ROI_ID`)
ALTER TABLE gistic_to_gene ADD CONSTRAINT fk_gistic_to_gene_GISTIC_ROI_ID FOREIGN KEY(GISTIC_ROI_ID) REFERENCES [gistic](GISTIC_ROI_ID)

ALTER TABLE [text_cache] ADD CONSTRAINT pk_text_cache PRIMARY KEY ([HASH_KEY])

ALTER TABLE [pfam_graphics] ADD CONSTRAINT pk_pfam_graphics PRIMARY KEY ([UNIPROT_ACC])

ALTER TABLE [drug] ADD CONSTRAINT pk_drug PRIMARY KEY ([DRUG_ID])



 -- TODO DEFINE: SINGLEKEY `DRUG_NAME` (`DRUG_NAME`)
CREATE INDEX ix_drug_DRUG_NAME ON drug([DRUG_NAME])

-- TODO DEFINE: FOREIGNKEY (`DRUG`) REFERENCES `drug` (`DRUG_ID`)
ALTER TABLE drug_interaction ADD CONSTRAINT fk_drug_interaction_DRUG FOREIGN KEY(DRUG) REFERENCES [drug](DRUG_ID)

ALTER TABLE [cna_event] ADD CONSTRAINT pk_cna_event PRIMARY KEY ([CNA_EVENT_ID])


-- TODO DEFINE: FOREIGNKEY (`ENTREZ_GENE_ID`) REFERENCES `gene` (`ENTREZ_GENE_ID`)
ALTER TABLE cna_event ADD CONSTRAINT fk_cna_event_ENTREZ_GENE_ID FOREIGN KEY(ENTREZ_GENE_ID) REFERENCES [gene](ENTREZ_GENE_ID)

ALTER TABLE [sample_cna_event] ADD CONSTRAINT pk_sample_cna_event PRIMARY KEY ([CNA_EVENT_ID],[SAMPLE_ID],[GENETIC_PROFILE_ID])



 -- TODO DEFINE: SINGLEKEY (`GENETIC_PROFILE_ID`,`SAMPLE_ID`)
CREATE INDEX ix_sample_cna_event_GENETIC_PROFILE_ID_SAMPLE_ID ON sample_cna_event([GENETIC_PROFILE_ID],[SAMPLE_ID])

-- TODO DEFINE: FOREIGNKEY (`CNA_EVENT_ID`) REFERENCES `cna_event` (`CNA_EVENT_ID`)
ALTER TABLE sample_cna_event ADD CONSTRAINT fk_sample_cna_event_CNA_EVENT_ID FOREIGN KEY(CNA_EVENT_ID) REFERENCES [cna_event](CNA_EVENT_ID)


-- TODO DEFINE: FOREIGNKEY (`GENETIC_PROFILE_ID`) REFERENCES `genetic_profile` (`GENETIC_PROFILE_ID`) ON DELETE CASCADE
ALTER TABLE sample_cna_event ADD CONSTRAINT fk_sample_cna_event_GENETIC_PROFILE_ID FOREIGN KEY(GENETIC_PROFILE_ID) REFERENCES [genetic_profile](GENETIC_PROFILE_ID)


-- TODO DEFINE: FOREIGNKEY (`SAMPLE_ID`) REFERENCES `sample` (`INTERNAL_ID`) ON DELETE CASCADE
ALTER TABLE sample_cna_event ADD CONSTRAINT fk_sample_cna_event_SAMPLE_ID FOREIGN KEY(SAMPLE_ID) REFERENCES [sample](INTERNAL_ID)

ALTER TABLE [copy_number_seg] ADD CONSTRAINT pk_copy_number_seg PRIMARY KEY ([SEG_ID])



 -- TODO DEFINE: SINGLEKEY (`CANCER_STUDY_ID`,`SAMPLE_ID`)
CREATE INDEX ix_copy_number_seg_CANCER_STUDY_ID_SAMPLE_ID ON copy_number_seg([CANCER_STUDY_ID],[SAMPLE_ID])

-- TODO DEFINE: FOREIGNKEY (`CANCER_STUDY_ID`) REFERENCES `cancer_study` (`CANCER_STUDY_ID`) ON DELETE CASCADE
ALTER TABLE copy_number_seg ADD CONSTRAINT fk_copy_number_seg_CANCER_STUDY_ID FOREIGN KEY(CANCER_STUDY_ID) REFERENCES [cancer_study](CANCER_STUDY_ID)


-- TODO DEFINE: FOREIGNKEY (`SAMPLE_ID`) REFERENCES `sample` (`INTERNAL_ID`) ON DELETE CASCADE
ALTER TABLE copy_number_seg ADD CONSTRAINT fk_copy_number_seg_SAMPLE_ID FOREIGN KEY(SAMPLE_ID) REFERENCES [sample](INTERNAL_ID)

ALTER TABLE [copy_number_seg_file] ADD CONSTRAINT pk_copy_number_seg_file PRIMARY KEY ([SEG_FILE_ID])


-- TODO DEFINE: FOREIGNKEY (`CANCER_STUDY_ID`) REFERENCES `cancer_study` (`CANCER_STUDY_ID`) ON DELETE CASCADE
ALTER TABLE copy_number_seg_file ADD CONSTRAINT fk_copy_number_seg_file_CANCER_STUDY_ID FOREIGN KEY(CANCER_STUDY_ID) REFERENCES [cancer_study](CANCER_STUDY_ID)

ALTER TABLE [cosmic_mutation] ADD CONSTRAINT pk_cosmic_mutation PRIMARY KEY ([COSMIC_MUTATION_ID])



 -- TODO DEFINE: SINGLEKEY (`KEYWORD`)
CREATE INDEX ix_cosmic_mutation_KEYWORD ON cosmic_mutation([KEYWORD])

-- TODO DEFINE: FOREIGNKEY (`ENTREZ_GENE_ID`) REFERENCES `gene` (`ENTREZ_GENE_ID`)
ALTER TABLE cosmic_mutation ADD CONSTRAINT fk_cosmic_mutation_ENTREZ_GENE_ID FOREIGN KEY(ENTREZ_GENE_ID) REFERENCES [gene](ENTREZ_GENE_ID)

ALTER TABLE [pdb_uniprot_alignment] ADD CONSTRAINT pk_pdb_uniprot_alignment PRIMARY KEY ([ALIGNMENT_ID])



 -- TODO DEFINE: SINGLEKEY(`UNIPROT_ID`)
CREATE INDEX ix_pdb_uniprot_alignment_UNIPROT_ID ON pdb_uniprot_alignment([UNIPROT_ID])


 -- TODO DEFINE: SINGLEKEY(`PDB_ID`, `CHAIN`)
CREATE INDEX ix_pdb_uniprot_alignment_PDB_ID_CHAIN ON pdb_uniprot_alignment([PDB_ID],[CHAIN])


 -- TODO DEFINE: SINGLEKEY(`ALIGNMENT_ID`, `UNIPROT_POSITION`)
CREATE INDEX ix_pdb_uniprot_residue_mapping_ALIGNMENT_ID_UNIPROT_POSITION ON pdb_uniprot_residue_mapping([ALIGNMENT_ID],[UNIPROT_POSITION])

-- TODO DEFINE: FOREIGNKEY(`ALIGNMENT_ID`) REFERENCES `pdb_uniprot_alignment` (`ALIGNMENT_ID`)
ALTER TABLE pdb_uniprot_residue_mapping ADD CONSTRAINT fk_pdb_uniprot_residue_mapping_ALIGNMENT_ID FOREIGN KEY(ALIGNMENT_ID) REFERENCES [pdb_uniprot_alignment](ALIGNMENT_ID)

ALTER TABLE [clinical_event] ADD CONSTRAINT pk_clinical_event PRIMARY KEY ([CLINICAL_EVENT_ID])



 -- TODO DEFINE: SINGLEKEY (`PATIENT_ID`, `EVENT_TYPE`)
CREATE INDEX ix_clinical_event_PATIENT_ID_EVENT_TYPE ON clinical_event([PATIENT_ID],[EVENT_TYPE])

-- TODO DEFINE: FOREIGNKEY (`PATIENT_ID`) REFERENCES `patient` (`INTERNAL_ID`) ON DELETE CASCADE
ALTER TABLE clinical_event ADD CONSTRAINT fk_clinical_event_PATIENT_ID FOREIGN KEY(PATIENT_ID) REFERENCES [patient](INTERNAL_ID)


-- TODO DEFINE: FOREIGNKEY (`CLINICAL_EVENT_ID`) REFERENCES `clinical_event` (`CLINICAL_EVENT_ID`) ON DELETE CASCADE
ALTER TABLE clinical_event_data ADD CONSTRAINT fk_clinical_event_data_CLINICAL_EVENT_ID FOREIGN KEY(CLINICAL_EVENT_ID) REFERENCES [clinical_event](CLINICAL_EVENT_ID)



GO

PRINT('LOADING TABLE type_of_cancer')
INSERT INTO "type_of_cancer" ("TYPE_OF_CANCER_ID","NAME","CLINICAL_TRIAL_KEYWORDS","DEDICATED_COLOR","SHORT_NAME","PARENT")
  VALUES ('brca','Breast Invasive Carcinoma','breast,breast invasive','HotPink','Breast','tissue');
INSERT INTO "type_of_cancer" ("TYPE_OF_CANCER_ID","NAME","CLINICAL_TRIAL_KEYWORDS","DEDICATED_COLOR","SHORT_NAME","PARENT")
  VALUES ('acc','Adrenocortical Carcinoma','adrenocortical carcinoma','Purple','ACC','adrenal_gland');

PRINT('LOADING TABLE cancer_study')
SET IDENTITY_INSERT [cancer_study] ON
INSERT INTO "cancer_study" ("CANCER_STUDY_ID", "CANCER_STUDY_IDENTIFIER", "TYPE_OF_CANCER_ID", "NAME", "SHORT_NAME", "DESCRIPTION", "PUBLIC", "PMID", "CITATION", "GROUPS", "STATUS", "IMPORT_DATE")
  VALUES (1,'study_tcga_pub','brca','Breast Invasive Carcinoma (TCGA, Nature 2012)','BRCA (TCGA)','<a href=\"http://cancergenome.nih.gov/\">The Cancer Genome Atlas (TCGA)</a> Breast Invasive Carcinoma project. 825 cases.<br><i>Nature 2012.</i> <a href=\"http://tcga-data.nci.nih.gov/tcga/\">Raw data via the TCGA Data Portal</a>.',1,'23000897','TCGA, Nature 2012','SU2C-PI3K;PUBLIC;GDAC', 0, '2011-12-18 13:17:17'); --//JK-UPDAtED
INSERT INTO "cancer_study" ("CANCER_STUDY_ID", "CANCER_STUDY_IDENTIFIER", "TYPE_OF_CANCER_ID", "NAME", "SHORT_NAME", "DESCRIPTION", "PUBLIC", "PMID", "CITATION", "GROUPS", "STATUS", "IMPORT_DATE")
  VALUES (2,'acc_tcga','acc','Adrenocortical Carcinoma (TCGA, Provisional)','ACC (TCGA)','TCGA Adrenocortical Carcinoma; raw data at the <A HREF="https://tcga-data.nci.nih.gov/">NCI</A>.',1,'23000897','TCGA, Nature 2012','SU2C-PI3K;PUBLIC;GDAC', 0, '2013-10-12 11:11:15');--//JK-UPDATED
SET IDENTITY_INSERT [cancer_study] OFF
  
PRINT('LOADING TABLE gene')
INSERT INTO "gene" ("ENTREZ_GENE_ID","HUGO_GENE_SYMBOL","TYPE","CYTOBAND","LENGTH") VALUES (207,'AKT1','protein-coding','14q32.32',10838);
INSERT INTO "gene" ("ENTREZ_GENE_ID","HUGO_GENE_SYMBOL","TYPE","CYTOBAND","LENGTH") VALUES (208,'AKT2','protein-coding','19q13.1-q13.2',15035);
INSERT INTO "gene" ("ENTREZ_GENE_ID","HUGO_GENE_SYMBOL","TYPE","CYTOBAND","LENGTH") VALUES (10000,'AKT3','protein-coding','1q44',7499);
INSERT INTO "gene" ("ENTREZ_GENE_ID","HUGO_GENE_SYMBOL","TYPE","CYTOBAND","LENGTH") VALUES (369,'ARAF','protein-coding','Xp11.4-p11.2',3204);
INSERT INTO "gene" ("ENTREZ_GENE_ID","HUGO_GENE_SYMBOL","TYPE","CYTOBAND","LENGTH") VALUES (472,'ATM','protein-coding','11q22-q23',22317);
INSERT INTO "gene" ("ENTREZ_GENE_ID","HUGO_GENE_SYMBOL","TYPE","CYTOBAND","LENGTH") VALUES (673,'BRAF','protein-coding','7q34',4564);
INSERT INTO "gene" ("ENTREZ_GENE_ID","HUGO_GENE_SYMBOL","TYPE","CYTOBAND","LENGTH") VALUES (672,'BRCA1','protein-coding','17q21',8426);
INSERT INTO "gene" ("ENTREZ_GENE_ID","HUGO_GENE_SYMBOL","TYPE","CYTOBAND","LENGTH") VALUES (675,'BRCA2','protein-coding','13q12.3',11269);
INSERT INTO "gene" ("ENTREZ_GENE_ID","HUGO_GENE_SYMBOL","TYPE","CYTOBAND","LENGTH") VALUES (3265,'HRAS','protein-coding','11p15.5',1854);
INSERT INTO "gene" ("ENTREZ_GENE_ID","HUGO_GENE_SYMBOL","TYPE","CYTOBAND","LENGTH") VALUES (3845,'KRAS','protein-coding','12p12.1',7302);
INSERT INTO "gene" ("ENTREZ_GENE_ID","HUGO_GENE_SYMBOL","TYPE","CYTOBAND","LENGTH") VALUES (4893,'NRAS','protein-coding','1p13.2',4449);
INSERT INTO "gene" ("ENTREZ_GENE_ID","HUGO_GENE_SYMBOL","TYPE","CYTOBAND","LENGTH") VALUES (79501,'OR4F5','protein-coding','1p36.33',918);
INSERT INTO "gene" ("ENTREZ_GENE_ID","HUGO_GENE_SYMBOL","TYPE","CYTOBAND","LENGTH") VALUES (148398,'SAMD11','protein-coding','1p36.33',3205);
INSERT INTO "gene" ("ENTREZ_GENE_ID","HUGO_GENE_SYMBOL","TYPE","CYTOBAND","LENGTH") VALUES (26155,'NOC2L','protein-coding','1p36.33',5539);
INSERT INTO "gene" ("ENTREZ_GENE_ID","HUGO_GENE_SYMBOL","TYPE","CYTOBAND","LENGTH") VALUES (2064,'ERBB2','protein-coding','17q12',10321);
INSERT INTO "gene" ("ENTREZ_GENE_ID","HUGO_GENE_SYMBOL","TYPE","CYTOBAND","LENGTH") VALUES (2886,'GRB7','protein-coding','17q12',3597);


-- cosmic_mutation
PRINT('LOADING TABLE cosmic_mutation')
INSERT INTO "cosmic_mutation" ("COSMIC_MUTATION_ID", "PROTEIN_CHANGE", "ENTREZ_GENE_ID", "COUNT", "KEYWORD") VALUES (3677745, 'D45A', 79501, 1, 'OR4F5 D45 missense');
INSERT INTO "cosmic_mutation" ("COSMIC_MUTATION_ID", "PROTEIN_CHANGE", "ENTREZ_GENE_ID", "COUNT", "KEYWORD") VALUES (426644, 'G145C', 79501, 1, 'OR4F5 G145 missense');
INSERT INTO "cosmic_mutation" ("COSMIC_MUTATION_ID", "PROTEIN_CHANGE", "ENTREZ_GENE_ID", "COUNT", "KEYWORD") VALUES (460103, 'P23P', 148398, 1, 'SAMD11 P23 silent');
INSERT INTO "cosmic_mutation" ("COSMIC_MUTATION_ID", "PROTEIN_CHANGE", "ENTREZ_GENE_ID", "COUNT", "KEYWORD") VALUES (4010395, 'S146S', 26155, 1, 'NOC2L S146 silent');
INSERT INTO "cosmic_mutation" ("COSMIC_MUTATION_ID", "PROTEIN_CHANGE", "ENTREZ_GENE_ID", "COUNT", "KEYWORD") VALUES (1290240, 'M1T', 26155, 1, 'NOC2L truncating');
INSERT INTO "cosmic_mutation" ("COSMIC_MUTATION_ID", "PROTEIN_CHANGE", "ENTREZ_GENE_ID", "COUNT", "KEYWORD") VALUES (4010425, 'Q197*', 26155, 1, 'NOC2L truncating');


PRINT('LOADING TABLE gene_alias')
INSERT INTO "gene_alias" ("ENTREZ_GENE_ID", "GENE_ALIAS") VALUES (207, 'AKT alias');
INSERT INTO "gene_alias" ("ENTREZ_GENE_ID", "GENE_ALIAS") VALUES (207, 'AKT alias2');
INSERT INTO "gene_alias" ("ENTREZ_GENE_ID", "GENE_ALIAS") VALUES (675, 'BRCA1 alias');

PRINT('LOADING TABLE genetic_profile')
SET IDENTITY_INSERT [genetic_profile] ON
INSERT INTO "genetic_profile" ("GENETIC_PROFILE_ID", "STABLE_ID", "CANCER_STUDY_ID", "GENETIC_ALTERATION_TYPE", "DATATYPE", "NAME", "DESCRIPTION", "SHOW_PROFILE_IN_ANALYSIS_TAB") VALUES (2,'study_tcga_pub_gistic',1,'COPY_NUMBER_ALTERATION','DISCRETE','Putative copy-number alterations from GISTIC','Putative copy-number from GISTIC 2.0. Values: -2 = homozygous deletion; -1 = hemizygous deletion; 0 = neutral / no change; 1 = gain; 2 = high level amplification.',1);
INSERT INTO "genetic_profile" ("GENETIC_PROFILE_ID", "STABLE_ID", "CANCER_STUDY_ID", "GENETIC_ALTERATION_TYPE", "DATATYPE", "NAME", "DESCRIPTION", "SHOW_PROFILE_IN_ANALYSIS_TAB") VALUES (3,'study_tcga_pub_mrna',1,'MRNA_EXPRESSION','Z-SCORE','mRNA expression (microarray)','Expression levels (Agilent microarray).',0);
INSERT INTO "genetic_profile" ("GENETIC_PROFILE_ID", "STABLE_ID", "CANCER_STUDY_ID", "GENETIC_ALTERATION_TYPE", "DATATYPE", "NAME", "DESCRIPTION", "SHOW_PROFILE_IN_ANALYSIS_TAB") VALUES (4,'study_tcga_pub_log2CNA',1,'COPY_NUMBER_ALTERATION','LOG-VALUE','Log2 copy-number values','Log2 copy-number values for each gene (from Affymetrix SNP6).',0);
INSERT INTO "genetic_profile" ("GENETIC_PROFILE_ID", "STABLE_ID", "CANCER_STUDY_ID", "GENETIC_ALTERATION_TYPE", "DATATYPE", "NAME", "DESCRIPTION", "SHOW_PROFILE_IN_ANALYSIS_TAB") VALUES (5,'study_tcga_pub_methylation_hm27',1,'METHYLATION','CONTINUOUS','Methylation (HM27)','Methylation beta-values (HM27 platform). For genes with multiple methylation probes, the probe least correlated with expression is selected.',0);
INSERT INTO "genetic_profile" ("GENETIC_PROFILE_ID", "STABLE_ID", "CANCER_STUDY_ID", "GENETIC_ALTERATION_TYPE", "DATATYPE", "NAME", "DESCRIPTION", "SHOW_PROFILE_IN_ANALYSIS_TAB") VALUES (6,'study_tcga_pub_mutations',1,'MUTATION_EXTENDED','MAF','Mutations','Mutation data from whole exome sequencing.',1);
INSERT INTO "genetic_profile" ("GENETIC_PROFILE_ID", "STABLE_ID", "CANCER_STUDY_ID", "GENETIC_ALTERATION_TYPE", "DATATYPE", "NAME", "DESCRIPTION", "SHOW_PROFILE_IN_ANALYSIS_TAB") VALUES (7,'study_tcga_pub_sv',1,'STRUCTURAL_VARIANT','SV','Structural Variants','Structural Variants detected by Illumina HiSeq sequencing.',1);
SET IDENTITY_INSERT [genetic_profile] OFF

-- genetic_profile_samples
PRINT('LOADING TABLE genetic_profile_samples')
INSERT INTO "genetic_profile_samples" ("GENETIC_PROFILE_ID", "ORDERED_SAMPLE_LIST") VALUES (2,'1,2,3,4,5,6,7,8,9,10,11,12,13,14,');
INSERT INTO "genetic_profile_samples" ("GENETIC_PROFILE_ID", "ORDERED_SAMPLE_LIST") VALUES (3,'2,3,6,8,9,10,12,13,');
INSERT INTO "genetic_profile_samples" ("GENETIC_PROFILE_ID", "ORDERED_SAMPLE_LIST") VALUES (4,'1,2,3,4,5,6,7,8,9,10,11,12,13,14,');
INSERT INTO "genetic_profile_samples" ("GENETIC_PROFILE_ID", "ORDERED_SAMPLE_LIST") VALUES (5,'2,');

-- patient
PRINT('LOADING TABLE patient')
SET IDENTITY_INSERT [patient] ON
INSERT INTO "patient" ("INTERNAL_ID", "STABLE_ID", "CANCER_STUDY_ID") VALUES (1,'TCGA-A1-A0SB',1);
INSERT INTO "patient" ("INTERNAL_ID", "STABLE_ID", "CANCER_STUDY_ID") VALUES (2,'TCGA-A1-A0SD',1);
INSERT INTO "patient" ("INTERNAL_ID", "STABLE_ID", "CANCER_STUDY_ID") VALUES (3,'TCGA-A1-A0SE',1);
INSERT INTO "patient" ("INTERNAL_ID", "STABLE_ID", "CANCER_STUDY_ID") VALUES (4,'TCGA-A1-A0SF',1);
INSERT INTO "patient" ("INTERNAL_ID", "STABLE_ID", "CANCER_STUDY_ID") VALUES (5,'TCGA-A1-A0SG',1);
INSERT INTO "patient" ("INTERNAL_ID", "STABLE_ID", "CANCER_STUDY_ID") VALUES (6,'TCGA-A1-A0SH',1);
INSERT INTO "patient" ("INTERNAL_ID", "STABLE_ID", "CANCER_STUDY_ID") VALUES (7,'TCGA-A1-A0SI',1);
INSERT INTO "patient" ("INTERNAL_ID", "STABLE_ID", "CANCER_STUDY_ID") VALUES (8,'TCGA-A1-A0SJ',1);
INSERT INTO "patient" ("INTERNAL_ID", "STABLE_ID", "CANCER_STUDY_ID") VALUES (9,'TCGA-A1-A0SK',1);
INSERT INTO "patient" ("INTERNAL_ID", "STABLE_ID", "CANCER_STUDY_ID") VALUES (10,'TCGA-A1-A0SM',1);
INSERT INTO "patient" ("INTERNAL_ID", "STABLE_ID", "CANCER_STUDY_ID") VALUES (11,'TCGA-A1-A0SN',1);
INSERT INTO "patient" ("INTERNAL_ID", "STABLE_ID", "CANCER_STUDY_ID") VALUES (12,'TCGA-A1-A0SO',1);
INSERT INTO "patient" ("INTERNAL_ID", "STABLE_ID", "CANCER_STUDY_ID") VALUES (13,'TCGA-A1-A0SP',1);
INSERT INTO "patient" ("INTERNAL_ID", "STABLE_ID", "CANCER_STUDY_ID") VALUES (14,'TCGA-A1-A0SQ',1);
INSERT INTO "patient" ("INTERNAL_ID", "STABLE_ID", "CANCER_STUDY_ID") VALUES (15,'TCGA-A1-B0SO',2);
INSERT INTO "patient" ("INTERNAL_ID", "STABLE_ID", "CANCER_STUDY_ID") VALUES (16,'TCGA-A1-B0SP',2);
INSERT INTO "patient" ("INTERNAL_ID", "STABLE_ID", "CANCER_STUDY_ID") VALUES (17,'TCGA-A1-B0SQ',2);
SET IDENTITY_INSERT [patient] OFF

-- sample
PRINT('LOADING TABLE sample')
SET IDENTITY_INSERT [sample] ON
INSERT INTO "sample" ("INTERNAL_ID","STABLE_ID","SAMPLE_TYPE","PATIENT_ID","TYPE_OF_CANCER_ID") VALUES (1,'TCGA-A1-A0SB-01','Primary Solid Tumor',1,'brca');
INSERT INTO "sample" ("INTERNAL_ID","STABLE_ID","SAMPLE_TYPE","PATIENT_ID","TYPE_OF_CANCER_ID") VALUES (2,'TCGA-A1-A0SD-01','Primary Solid Tumor',2,'brca');
INSERT INTO "sample" ("INTERNAL_ID","STABLE_ID","SAMPLE_TYPE","PATIENT_ID","TYPE_OF_CANCER_ID") VALUES (3,'TCGA-A1-A0SE-01','Primary Solid Tumor',3,'brca');
INSERT INTO "sample" ("INTERNAL_ID","STABLE_ID","SAMPLE_TYPE","PATIENT_ID","TYPE_OF_CANCER_ID") VALUES (4,'TCGA-A1-A0SF-01','Primary Solid Tumor',4,'brca');
INSERT INTO "sample" ("INTERNAL_ID","STABLE_ID","SAMPLE_TYPE","PATIENT_ID","TYPE_OF_CANCER_ID") VALUES (5,'TCGA-A1-A0SG-01','Primary Solid Tumor',5,'brca');
INSERT INTO "sample" ("INTERNAL_ID","STABLE_ID","SAMPLE_TYPE","PATIENT_ID","TYPE_OF_CANCER_ID") VALUES (6,'TCGA-A1-A0SH-01','Primary Solid Tumor',6,'brca');
INSERT INTO "sample" ("INTERNAL_ID","STABLE_ID","SAMPLE_TYPE","PATIENT_ID","TYPE_OF_CANCER_ID") VALUES (7,'TCGA-A1-A0SI-01','Primary Solid Tumor',7,'brca');
INSERT INTO "sample" ("INTERNAL_ID","STABLE_ID","SAMPLE_TYPE","PATIENT_ID","TYPE_OF_CANCER_ID") VALUES (8,'TCGA-A1-A0SJ-01','Primary Solid Tumor',8,'brca');
INSERT INTO "sample" ("INTERNAL_ID","STABLE_ID","SAMPLE_TYPE","PATIENT_ID","TYPE_OF_CANCER_ID") VALUES (9,'TCGA-A1-A0SK-01','Primary Solid Tumor',9,'brca');
INSERT INTO "sample" ("INTERNAL_ID","STABLE_ID","SAMPLE_TYPE","PATIENT_ID","TYPE_OF_CANCER_ID") VALUES (10,'TCGA-A1-A0SM-01','Primary Solid Tumor',10,'brca');
INSERT INTO "sample" ("INTERNAL_ID","STABLE_ID","SAMPLE_TYPE","PATIENT_ID","TYPE_OF_CANCER_ID") VALUES (11,'TCGA-A1-A0SN-01','Primary Solid Tumor',11,'brca');
INSERT INTO "sample" ("INTERNAL_ID","STABLE_ID","SAMPLE_TYPE","PATIENT_ID","TYPE_OF_CANCER_ID") VALUES (12,'TCGA-A1-A0SO-01','Primary Solid Tumor',12,'brca');
INSERT INTO "sample" ("INTERNAL_ID","STABLE_ID","SAMPLE_TYPE","PATIENT_ID","TYPE_OF_CANCER_ID") VALUES (13,'TCGA-A1-A0SP-01','Primary Solid Tumor',13,'brca');
INSERT INTO "sample" ("INTERNAL_ID","STABLE_ID","SAMPLE_TYPE","PATIENT_ID","TYPE_OF_CANCER_ID") VALUES (14,'TCGA-A1-A0SQ-01','Primary Solid Tumor',14,'brca');
INSERT INTO "sample" ("INTERNAL_ID","STABLE_ID","SAMPLE_TYPE","PATIENT_ID","TYPE_OF_CANCER_ID") VALUES (15,'TCGA-A1-B0SO-01','Primary Solid Tumor',15,'acc');
INSERT INTO "sample" ("INTERNAL_ID","STABLE_ID","SAMPLE_TYPE","PATIENT_ID","TYPE_OF_CANCER_ID") VALUES (16,'TCGA-A1-B0SP-01','Primary Solid Tumor',16,'acc');
INSERT INTO "sample" ("INTERNAL_ID","STABLE_ID","SAMPLE_TYPE","PATIENT_ID","TYPE_OF_CANCER_ID") VALUES (17,'TCGA-A1-B0SQ-01','Primary Solid Tumor',17,'acc');
INSERT INTO "sample" ("INTERNAL_ID","STABLE_ID","SAMPLE_TYPE","PATIENT_ID","TYPE_OF_CANCER_ID") VALUES (18,'TCGA-A1-A0SB-02','Primary Solid Tumor',1,'brca');
SET IDENTITY_INSERT [sample] OFF


-- mutation_event
PRINT('LOADING TABLE mutation_event')
SET IDENTITY_INSERT [mutation_event] ON
INSERT INTO "mutation_event" ("MUTATION_EVENT_ID","ENTREZ_GENE_ID","CHR","START_POSITION","END_POSITION","REFERENCE_ALLELE","TUMOR_SEQ_ALLELE","PROTEIN_CHANGE","MUTATION_TYPE","FUNCTIONAL_IMPACT_SCORE","FIS_VALUE","LINK_XVAR","LINK_PDB","LINK_MSA","NCBI_BUILD","STRAND","VARIANT_TYPE","DB_SNP_RS","DB_SNP_VAL_STATUS","ONCOTATOR_DBSNP_RS","ONCOTATOR_REFSEQ_MRNA_ID","ONCOTATOR_CODON_CHANGE","ONCOTATOR_UNIPROT_ENTRY_NAME","ONCOTATOR_UNIPROT_ACCESSION","ONCOTATOR_PROTEIN_POS_START","ONCOTATOR_PROTEIN_POS_END","CANONICAL_TRANSCRIPT","KEYWORD") VALUES (2038,672,'17',41244748,41244748,'G','A','Q934*','Nonsense_Mutation','NA',0,'getma.org/?cm=var&var=hg19,17,41244748,G,A&fts=all','NA','NA','37','+','SNP','rs80357223','unknown','rs80357223','NM_007294','c.(2800-2802)CAG>TAG','BRCA1_HUMAN','P38398',934,934,1,'BRCA1 truncating');
INSERT INTO "mutation_event" ("MUTATION_EVENT_ID","ENTREZ_GENE_ID","CHR","START_POSITION","END_POSITION","REFERENCE_ALLELE","TUMOR_SEQ_ALLELE","PROTEIN_CHANGE","MUTATION_TYPE","FUNCTIONAL_IMPACT_SCORE","FIS_VALUE","LINK_XVAR","LINK_PDB","LINK_MSA","NCBI_BUILD","STRAND","VARIANT_TYPE","DB_SNP_RS","DB_SNP_VAL_STATUS","ONCOTATOR_DBSNP_RS","ONCOTATOR_REFSEQ_MRNA_ID","ONCOTATOR_CODON_CHANGE","ONCOTATOR_UNIPROT_ENTRY_NAME","ONCOTATOR_UNIPROT_ACCESSION","ONCOTATOR_PROTEIN_POS_START","ONCOTATOR_PROTEIN_POS_END","CANONICAL_TRANSCRIPT","KEYWORD") VALUES (22604,672,'17',41258504,41258504,'A','C','C61G','Missense_Mutation','H',4.355,'getma.org/?cm=var&var=hg19,17,41258504,A,C&fts=all','getma.org/pdb.php?prot=BRCA1_HUMAN&from=24&to=64&var=C61G','getma.org/?cm=msa&ty=f&p=BRCA1_HUMAN&rb=24&re=64&var=C61G','37','+','SNP','rs28897672','byCluster','rs28897672','NM_007294','c.(181-183)TGT>GGT','BRCA1_HUMAN','P38398',61,61,1,'BRCA1 C61 missense');
INSERT INTO "mutation_event" ("MUTATION_EVENT_ID","ENTREZ_GENE_ID","CHR","START_POSITION","END_POSITION","REFERENCE_ALLELE","TUMOR_SEQ_ALLELE","PROTEIN_CHANGE","MUTATION_TYPE","FUNCTIONAL_IMPACT_SCORE","FIS_VALUE","LINK_XVAR","LINK_PDB","LINK_MSA","NCBI_BUILD","STRAND","VARIANT_TYPE","DB_SNP_RS","DB_SNP_VAL_STATUS","ONCOTATOR_DBSNP_RS","ONCOTATOR_REFSEQ_MRNA_ID","ONCOTATOR_CODON_CHANGE","ONCOTATOR_UNIPROT_ENTRY_NAME","ONCOTATOR_UNIPROT_ACCESSION","ONCOTATOR_PROTEIN_POS_START","ONCOTATOR_PROTEIN_POS_END","CANONICAL_TRANSCRIPT","KEYWORD") VALUES (2039,672,'17',41276033,41276033,'C','T','C27_splice','Splice_Site','NA',1.4013e-45,'NA','NA','NA','37','+','SNP','rs80358010','byCluster','rs80358010','NM_007294','c.e2+1','NA','NA',-1,-1,1,'BRCA1 truncating');
INSERT INTO "mutation_event" ("MUTATION_EVENT_ID","ENTREZ_GENE_ID","CHR","START_POSITION","END_POSITION","REFERENCE_ALLELE","TUMOR_SEQ_ALLELE","PROTEIN_CHANGE","MUTATION_TYPE","FUNCTIONAL_IMPACT_SCORE","FIS_VALUE","LINK_XVAR","LINK_PDB","LINK_MSA","NCBI_BUILD","STRAND","VARIANT_TYPE","DB_SNP_RS","DB_SNP_VAL_STATUS","ONCOTATOR_DBSNP_RS","ONCOTATOR_REFSEQ_MRNA_ID","ONCOTATOR_CODON_CHANGE","ONCOTATOR_UNIPROT_ENTRY_NAME","ONCOTATOR_UNIPROT_ACCESSION","ONCOTATOR_PROTEIN_POS_START","ONCOTATOR_PROTEIN_POS_END","CANONICAL_TRANSCRIPT","KEYWORD") VALUES (2040,207,'17',41244748,41244748,'G','A','Q934*','Nonsense_Mutation','NA',0,'getma.org/?cm=var&var=hg19,17,41244748,G,A&fts=all','NA','NA','37','+','SNP','rs80357223','unknown','rs80357223','NM_007294','c.(2800-2802)CAG>TAG','BRCA1_HUMAN','P38398',934,934,1,'BRCA1 truncating');
INSERT INTO "mutation_event" ("MUTATION_EVENT_ID","ENTREZ_GENE_ID","CHR","START_POSITION","END_POSITION","REFERENCE_ALLELE","TUMOR_SEQ_ALLELE","PROTEIN_CHANGE","MUTATION_TYPE","FUNCTIONAL_IMPACT_SCORE","FIS_VALUE","LINK_XVAR","LINK_PDB","LINK_MSA","NCBI_BUILD","STRAND","VARIANT_TYPE","DB_SNP_RS","DB_SNP_VAL_STATUS","ONCOTATOR_DBSNP_RS","ONCOTATOR_REFSEQ_MRNA_ID","ONCOTATOR_CODON_CHANGE","ONCOTATOR_UNIPROT_ENTRY_NAME","ONCOTATOR_UNIPROT_ACCESSION","ONCOTATOR_PROTEIN_POS_START","ONCOTATOR_PROTEIN_POS_END","CANONICAL_TRANSCRIPT","KEYWORD") VALUES (2041,207,'17',41258504,41258504,'A','C','C61G','Missense_Mutation','H',4.355,'getma.org/?cm=var&var=hg19,17,41258504,A,C&fts=all','getma.org/pdb.php?prot=BRCA1_HUMAN&from=24&to=64&var=C61G','getma.org/?cm=msa&ty=f&p=BRCA1_HUMAN&rb=24&re=64&var=C61G','37','+','SNP','rs28897672','byCluster','rs28897672','NM_007294','c.(181-183)TGT>GGT','BRCA1_HUMAN','P38398',61,61,1,'BRCA1 C61 missense');
INSERT INTO "mutation_event" ("MUTATION_EVENT_ID","ENTREZ_GENE_ID","CHR","START_POSITION","END_POSITION","REFERENCE_ALLELE","TUMOR_SEQ_ALLELE","PROTEIN_CHANGE","MUTATION_TYPE","FUNCTIONAL_IMPACT_SCORE","FIS_VALUE","LINK_XVAR","LINK_PDB","LINK_MSA","NCBI_BUILD","STRAND","VARIANT_TYPE","DB_SNP_RS","DB_SNP_VAL_STATUS","ONCOTATOR_DBSNP_RS","ONCOTATOR_REFSEQ_MRNA_ID","ONCOTATOR_CODON_CHANGE","ONCOTATOR_UNIPROT_ENTRY_NAME","ONCOTATOR_UNIPROT_ACCESSION","ONCOTATOR_PROTEIN_POS_START","ONCOTATOR_PROTEIN_POS_END","CANONICAL_TRANSCRIPT","KEYWORD") VALUES (2042,208,'17',41276033,41276033,'C','T','C27_splice','Splice_Site','NA',1.4013e-45,'NA','NA','NA','37','+','SNP','rs80358010','byCluster','rs80358010','NM_007294','c.e2+1','NA','NA',-1,-1,1,'BRCA1 truncating');
SET IDENTITY_INSERT [mutation_event] OFF

-- mutation
PRINT('LOADING TABLE mutation')
INSERT INTO "mutation" ("MUTATION_EVENT_ID","GENETIC_PROFILE_ID","SAMPLE_ID","ENTREZ_GENE_ID","CENTER","SEQUENCER","MUTATION_STATUS","VALIDATION_STATUS","TUMOR_SEQ_ALLELE1","TUMOR_SEQ_ALLELE2","MATCHED_NORM_SAMPLE_BARCODE","MATCH_NORM_SEQ_ALLELE1","MATCH_NORM_SEQ_ALLELE2","TUMOR_VALIDATION_ALLELE1","TUMOR_VALIDATION_ALLELE2","MATCH_NORM_VALIDATION_ALLELE1","MATCH_NORM_VALIDATION_ALLELE2","VERIFICATION_STATUS","SEQUENCING_PHASE","SEQUENCE_SOURCE","VALIDATION_METHOD","SCORE","BAM_FILE","TUMOR_ALT_COUNT","TUMOR_REF_COUNT","NORMAL_ALT_COUNT","NORMAL_REF_COUNT") VALUES (2038,6,6,672,'genome.wustl.edu','IlluminaGAIIx','Germline','Unknown','G','A','TCGA-A1-A0SH-10A-03D-A099-09','G','A','NA','NA','NA','NA','Unknown','Phase_IV','Capture','NA','1','dbGAP',1,0,-1,-1);
INSERT INTO "mutation" ("MUTATION_EVENT_ID","GENETIC_PROFILE_ID","SAMPLE_ID","ENTREZ_GENE_ID","CENTER","SEQUENCER","MUTATION_STATUS","VALIDATION_STATUS","TUMOR_SEQ_ALLELE1","TUMOR_SEQ_ALLELE2","MATCHED_NORM_SAMPLE_BARCODE","MATCH_NORM_SEQ_ALLELE1","MATCH_NORM_SEQ_ALLELE2","TUMOR_VALIDATION_ALLELE1","TUMOR_VALIDATION_ALLELE2","MATCH_NORM_VALIDATION_ALLELE1","MATCH_NORM_VALIDATION_ALLELE2","VERIFICATION_STATUS","SEQUENCING_PHASE","SEQUENCE_SOURCE","VALIDATION_METHOD","SCORE","BAM_FILE","TUMOR_ALT_COUNT","TUMOR_REF_COUNT","NORMAL_ALT_COUNT","NORMAL_REF_COUNT") VALUES (22604,6,6,672,'genome.wustl.edu','IlluminaGAIIx','Germline','Unknown','A','C','TCGA-A1-A0SH-10A-03D-A099-09','A','C','NA','NA','NA','NA','Unknown','Phase_IV','Capture','NA','1','dbGAP',-1,-1,-1,-1);
INSERT INTO "mutation" ("MUTATION_EVENT_ID","GENETIC_PROFILE_ID","SAMPLE_ID","ENTREZ_GENE_ID","CENTER","SEQUENCER","MUTATION_STATUS","VALIDATION_STATUS","TUMOR_SEQ_ALLELE1","TUMOR_SEQ_ALLELE2","MATCHED_NORM_SAMPLE_BARCODE","MATCH_NORM_SEQ_ALLELE1","MATCH_NORM_SEQ_ALLELE2","TUMOR_VALIDATION_ALLELE1","TUMOR_VALIDATION_ALLELE2","MATCH_NORM_VALIDATION_ALLELE1","MATCH_NORM_VALIDATION_ALLELE2","VERIFICATION_STATUS","SEQUENCING_PHASE","SEQUENCE_SOURCE","VALIDATION_METHOD","SCORE","BAM_FILE","TUMOR_ALT_COUNT","TUMOR_REF_COUNT","NORMAL_ALT_COUNT","NORMAL_REF_COUNT") VALUES (2039,6,12,672,'genome.wustl.edu','IlluminaGAIIx','Germline','Unknown','T','T','TCGA-A1-A0SO-10A-03D-A099-09','T','T','NA','NA','NA','NA','Unknown','Phase_IV','Capture','NA','1','dbGAP',-1,-1,-1,-1);
INSERT INTO "mutation" ("MUTATION_EVENT_ID","GENETIC_PROFILE_ID","SAMPLE_ID","ENTREZ_GENE_ID","CENTER","SEQUENCER","MUTATION_STATUS","VALIDATION_STATUS","TUMOR_SEQ_ALLELE1","TUMOR_SEQ_ALLELE2","MATCHED_NORM_SAMPLE_BARCODE","MATCH_NORM_SEQ_ALLELE1","MATCH_NORM_SEQ_ALLELE2","TUMOR_VALIDATION_ALLELE1","TUMOR_VALIDATION_ALLELE2","MATCH_NORM_VALIDATION_ALLELE1","MATCH_NORM_VALIDATION_ALLELE2","VERIFICATION_STATUS","SEQUENCING_PHASE","SEQUENCE_SOURCE","VALIDATION_METHOD","SCORE","BAM_FILE","TUMOR_ALT_COUNT","TUMOR_REF_COUNT","NORMAL_ALT_COUNT","NORMAL_REF_COUNT") VALUES (2038,6,7,672,'genome.wustl.edu','IlluminaGAIIx','Germline','Unknown','G','A','TCGA-A1-A0SH-10A-03D-A099-09','G','A','NA','NA','NA','NA','Unknown','Phase_IV','Capture','NA','1','dbGAP',-1,-1,-1,-1);
INSERT INTO "mutation" ("MUTATION_EVENT_ID","GENETIC_PROFILE_ID","SAMPLE_ID","ENTREZ_GENE_ID","CENTER","SEQUENCER","MUTATION_STATUS","VALIDATION_STATUS","TUMOR_SEQ_ALLELE1","TUMOR_SEQ_ALLELE2","MATCHED_NORM_SAMPLE_BARCODE","MATCH_NORM_SEQ_ALLELE1","MATCH_NORM_SEQ_ALLELE2","TUMOR_VALIDATION_ALLELE1","TUMOR_VALIDATION_ALLELE2","MATCH_NORM_VALIDATION_ALLELE1","MATCH_NORM_VALIDATION_ALLELE2","VERIFICATION_STATUS","SEQUENCING_PHASE","SEQUENCE_SOURCE","VALIDATION_METHOD","SCORE","BAM_FILE","TUMOR_ALT_COUNT","TUMOR_REF_COUNT","NORMAL_ALT_COUNT","NORMAL_REF_COUNT") VALUES (2038,6,7,672,'genome.wustl.edu','IlluminaGAIIx','Germline','Unknown','A','C','TCGA-A1-A0SH-10A-03D-A099-09','A','C','NA','NA','NA','NA','Unknown','Phase_IV','Capture','NA','1','dbGAP',-1,1,-1,-1);
INSERT INTO "mutation" ("MUTATION_EVENT_ID","GENETIC_PROFILE_ID","SAMPLE_ID","ENTREZ_GENE_ID","CENTER","SEQUENCER","MUTATION_STATUS","VALIDATION_STATUS","TUMOR_SEQ_ALLELE1","TUMOR_SEQ_ALLELE2","MATCHED_NORM_SAMPLE_BARCODE","MATCH_NORM_SEQ_ALLELE1","MATCH_NORM_SEQ_ALLELE2","TUMOR_VALIDATION_ALLELE1","TUMOR_VALIDATION_ALLELE2","MATCH_NORM_VALIDATION_ALLELE1","MATCH_NORM_VALIDATION_ALLELE2","VERIFICATION_STATUS","SEQUENCING_PHASE","SEQUENCE_SOURCE","VALIDATION_METHOD","SCORE","BAM_FILE","TUMOR_ALT_COUNT","TUMOR_REF_COUNT","NORMAL_ALT_COUNT","NORMAL_REF_COUNT") VALUES (2039,6,13,672,'genome.wustl.edu','IlluminaGAIIx','Germline','Unknown','T','T','TCGA-A1-A0SO-10A-03D-A099-09','T','T','NA','NA','NA','NA','Unknown','Phase_IV','Capture','NA','1','dbGAP',-1,-1,-1,-1);
INSERT INTO "mutation" ("MUTATION_EVENT_ID","GENETIC_PROFILE_ID","SAMPLE_ID","ENTREZ_GENE_ID","CENTER","SEQUENCER","MUTATION_STATUS","VALIDATION_STATUS","TUMOR_SEQ_ALLELE1","TUMOR_SEQ_ALLELE2","MATCHED_NORM_SAMPLE_BARCODE","MATCH_NORM_SEQ_ALLELE1","MATCH_NORM_SEQ_ALLELE2","TUMOR_VALIDATION_ALLELE1","TUMOR_VALIDATION_ALLELE2","MATCH_NORM_VALIDATION_ALLELE1","MATCH_NORM_VALIDATION_ALLELE2","VERIFICATION_STATUS","SEQUENCING_PHASE","SEQUENCE_SOURCE","VALIDATION_METHOD","SCORE","BAM_FILE","TUMOR_ALT_COUNT","TUMOR_REF_COUNT","NORMAL_ALT_COUNT","NORMAL_REF_COUNT") VALUES (2040,6,1,207,'genome.wustl.edu','IlluminaGAIIx','Germline','Unknown','G','A','TCGA-A1-A0SH-10A-03D-A099-09','G','A','NA','NA','NA','NA','Unknown','Phase_IV','Capture','NA','1','dbGAP',-1,-1,-1,-1);
INSERT INTO "mutation" ("MUTATION_EVENT_ID","GENETIC_PROFILE_ID","SAMPLE_ID","ENTREZ_GENE_ID","CENTER","SEQUENCER","MUTATION_STATUS","VALIDATION_STATUS","TUMOR_SEQ_ALLELE1","TUMOR_SEQ_ALLELE2","MATCHED_NORM_SAMPLE_BARCODE","MATCH_NORM_SEQ_ALLELE1","MATCH_NORM_SEQ_ALLELE2","TUMOR_VALIDATION_ALLELE1","TUMOR_VALIDATION_ALLELE2","MATCH_NORM_VALIDATION_ALLELE1","MATCH_NORM_VALIDATION_ALLELE2","VERIFICATION_STATUS","SEQUENCING_PHASE","SEQUENCE_SOURCE","VALIDATION_METHOD","SCORE","BAM_FILE","TUMOR_ALT_COUNT","TUMOR_REF_COUNT","NORMAL_ALT_COUNT","NORMAL_REF_COUNT") VALUES (2041,6,2,207,'genome.wustl.edu','IlluminaGAIIx','Germline','Unknown','A','C','TCGA-A1-A0SH-10A-03D-A099-09','A','C','NA','NA','NA','NA','Unknown','Phase_IV','Capture','NA','1','dbGAP',0,-1,-1,-1);
INSERT INTO "mutation" ("MUTATION_EVENT_ID","GENETIC_PROFILE_ID","SAMPLE_ID","ENTREZ_GENE_ID","CENTER","SEQUENCER","MUTATION_STATUS","VALIDATION_STATUS","TUMOR_SEQ_ALLELE1","TUMOR_SEQ_ALLELE2","MATCHED_NORM_SAMPLE_BARCODE","MATCH_NORM_SEQ_ALLELE1","MATCH_NORM_SEQ_ALLELE2","TUMOR_VALIDATION_ALLELE1","TUMOR_VALIDATION_ALLELE2","MATCH_NORM_VALIDATION_ALLELE1","MATCH_NORM_VALIDATION_ALLELE2","VERIFICATION_STATUS","SEQUENCING_PHASE","SEQUENCE_SOURCE","VALIDATION_METHOD","SCORE","BAM_FILE","TUMOR_ALT_COUNT","TUMOR_REF_COUNT","NORMAL_ALT_COUNT","NORMAL_REF_COUNT") VALUES (2042,6,3,208,'genome.wustl.edu','IlluminaGAIIx','Germline','Unknown','T','T','TCGA-A1-A0SO-10A-03D-A099-09','T','T','NA','NA','NA','NA','Unknown','Phase_IV','Capture','NA','1','dbGAP',-1,-1,-1,-1);

-- mutation_count
PRINT('LOADING TABLE mutation_count')
INSERT INTO "mutation_count" ("GENETIC_PROFILE_ID","SAMPLE_ID","MUTATION_COUNT") VALUES (6,1,1);
INSERT INTO "mutation_count" ("GENETIC_PROFILE_ID","SAMPLE_ID","MUTATION_COUNT") VALUES (6,2,1);
INSERT INTO "mutation_count" ("GENETIC_PROFILE_ID","SAMPLE_ID","MUTATION_COUNT") VALUES (6,3,1);
INSERT INTO "mutation_count" ("GENETIC_PROFILE_ID","SAMPLE_ID","MUTATION_COUNT") VALUES (6,6,2);
INSERT INTO "mutation_count" ("GENETIC_PROFILE_ID","SAMPLE_ID","MUTATION_COUNT") VALUES (6,7,2);
INSERT INTO "mutation_count" ("GENETIC_PROFILE_ID","SAMPLE_ID","MUTATION_COUNT") VALUES (6,12,1);
INSERT INTO "mutation_count" ("GENETIC_PROFILE_ID","SAMPLE_ID","MUTATION_COUNT") VALUES (6,13,1);


PRINT('LOADING TABLE gene_panel')
SET IDENTITY_INSERT [gene_panel] ON
INSERT INTO "gene_panel" ("INTERNAL_ID", "STABLE_ID", "DESCRIPTION") VALUES(1, 'TESTPANEL1', 'A test panel consisting of a few genes');
INSERT INTO "gene_panel" ("INTERNAL_ID", "STABLE_ID", "DESCRIPTION") VALUES(2, 'TESTPANEL2', 'Another test panel consisting of a few genes');
SET IDENTITY_INSERT [gene_panel] OFF

PRINT('LOADING TABLE gene_panel_list')
INSERT INTO "gene_panel_list" ("INTERNAL_ID", "GENE_ID") VALUES(1, 207);
INSERT INTO "gene_panel_list" ("INTERNAL_ID", "GENE_ID") VALUES(1, 369);
INSERT INTO "gene_panel_list" ("INTERNAL_ID", "GENE_ID") VALUES(1, 672);

INSERT INTO "gene_panel_list" ("INTERNAL_ID", "GENE_ID") VALUES(2, 207);
INSERT INTO "gene_panel_list" ("INTERNAL_ID", "GENE_ID") VALUES(2, 208);
INSERT INTO "gene_panel_list" ("INTERNAL_ID", "GENE_ID") VALUES(2, 4893);


-- sample_profile
PRINT('LOADING TABLE Sample_profile')
INSERT INTO "sample_profile" ("SAMPLE_ID","GENETIC_PROFILE_ID","PANEL_ID") VALUES (1,2,1);
INSERT INTO "sample_profile" ("SAMPLE_ID","GENETIC_PROFILE_ID","PANEL_ID") VALUES (1,3,1);
INSERT INTO "sample_profile" ("SAMPLE_ID","GENETIC_PROFILE_ID","PANEL_ID") VALUES (1,4,NULL);
INSERT INTO "sample_profile" ("SAMPLE_ID","GENETIC_PROFILE_ID","PANEL_ID") VALUES (2,2,2);
INSERT INTO "sample_profile" ("SAMPLE_ID","GENETIC_PROFILE_ID","PANEL_ID") VALUES (2,3,NULL);
INSERT INTO "sample_profile" ("SAMPLE_ID","GENETIC_PROFILE_ID","PANEL_ID") VALUES (2,4,2);
INSERT INTO "sample_profile" ("SAMPLE_ID","GENETIC_PROFILE_ID","PANEL_ID") VALUES (2,5,NULL);
INSERT INTO "sample_profile" ("SAMPLE_ID","GENETIC_PROFILE_ID","PANEL_ID") VALUES (2,6,NULL);
INSERT INTO "sample_profile" ("SAMPLE_ID","GENETIC_PROFILE_ID","PANEL_ID") VALUES (3,2,NULL);
INSERT INTO "sample_profile" ("SAMPLE_ID","GENETIC_PROFILE_ID","PANEL_ID") VALUES (3,3,NULL);
INSERT INTO "sample_profile" ("SAMPLE_ID","GENETIC_PROFILE_ID","PANEL_ID") VALUES (3,4,NULL);
INSERT INTO "sample_profile" ("SAMPLE_ID","GENETIC_PROFILE_ID","PANEL_ID") VALUES (3,6,NULL);
INSERT INTO "sample_profile" ("SAMPLE_ID","GENETIC_PROFILE_ID","PANEL_ID") VALUES (4,2,NULL);
INSERT INTO "sample_profile" ("SAMPLE_ID","GENETIC_PROFILE_ID","PANEL_ID") VALUES (4,4,NULL);
INSERT INTO "sample_profile" ("SAMPLE_ID","GENETIC_PROFILE_ID","PANEL_ID") VALUES (5,2,NULL);
INSERT INTO "sample_profile" ("SAMPLE_ID","GENETIC_PROFILE_ID","PANEL_ID") VALUES (5,4,NULL);
INSERT INTO "sample_profile" ("SAMPLE_ID","GENETIC_PROFILE_ID","PANEL_ID") VALUES (6,2,NULL);
INSERT INTO "sample_profile" ("SAMPLE_ID","GENETIC_PROFILE_ID","PANEL_ID") VALUES (6,3,NULL);
INSERT INTO "sample_profile" ("SAMPLE_ID","GENETIC_PROFILE_ID","PANEL_ID") VALUES (6,4,NULL);
INSERT INTO "sample_profile" ("SAMPLE_ID","GENETIC_PROFILE_ID","PANEL_ID") VALUES (6,6,NULL);
INSERT INTO "sample_profile" ("SAMPLE_ID","GENETIC_PROFILE_ID","PANEL_ID") VALUES (7,2,NULL);
INSERT INTO "sample_profile" ("SAMPLE_ID","GENETIC_PROFILE_ID","PANEL_ID") VALUES (7,4,NULL);
INSERT INTO "sample_profile" ("SAMPLE_ID","GENETIC_PROFILE_ID","PANEL_ID") VALUES (8,2,NULL);
INSERT INTO "sample_profile" ("SAMPLE_ID","GENETIC_PROFILE_ID","PANEL_ID") VALUES (8,3,NULL);
INSERT INTO "sample_profile" ("SAMPLE_ID","GENETIC_PROFILE_ID","PANEL_ID") VALUES (8,4,NULL);
INSERT INTO "sample_profile" ("SAMPLE_ID","GENETIC_PROFILE_ID","PANEL_ID") VALUES (8,6,NULL);
INSERT INTO "sample_profile" ("SAMPLE_ID","GENETIC_PROFILE_ID","PANEL_ID") VALUES (9,2,NULL);
INSERT INTO "sample_profile" ("SAMPLE_ID","GENETIC_PROFILE_ID","PANEL_ID") VALUES (9,3,NULL);
INSERT INTO "sample_profile" ("SAMPLE_ID","GENETIC_PROFILE_ID","PANEL_ID") VALUES (9,4,NULL);
INSERT INTO "sample_profile" ("SAMPLE_ID","GENETIC_PROFILE_ID","PANEL_ID") VALUES (9,6,NULL);
INSERT INTO "sample_profile" ("SAMPLE_ID","GENETIC_PROFILE_ID","PANEL_ID") VALUES (10,2,NULL);
INSERT INTO "sample_profile" ("SAMPLE_ID","GENETIC_PROFILE_ID","PANEL_ID") VALUES (10,3,NULL);
INSERT INTO "sample_profile" ("SAMPLE_ID","GENETIC_PROFILE_ID","PANEL_ID") VALUES (10,4,NULL);
INSERT INTO "sample_profile" ("SAMPLE_ID","GENETIC_PROFILE_ID","PANEL_ID") VALUES (10,6,NULL);
INSERT INTO "sample_profile" ("SAMPLE_ID","GENETIC_PROFILE_ID","PANEL_ID") VALUES (11,2,NULL);
INSERT INTO "sample_profile" ("SAMPLE_ID","GENETIC_PROFILE_ID","PANEL_ID") VALUES (11,4,NULL);
INSERT INTO "sample_profile" ("SAMPLE_ID","GENETIC_PROFILE_ID","PANEL_ID") VALUES (12,2,NULL);
INSERT INTO "sample_profile" ("SAMPLE_ID","GENETIC_PROFILE_ID","PANEL_ID") VALUES (12,3,NULL);
INSERT INTO "sample_profile" ("SAMPLE_ID","GENETIC_PROFILE_ID","PANEL_ID") VALUES (12,4,NULL);
INSERT INTO "sample_profile" ("SAMPLE_ID","GENETIC_PROFILE_ID","PANEL_ID") VALUES (12,6,NULL);
INSERT INTO "sample_profile" ("SAMPLE_ID","GENETIC_PROFILE_ID","PANEL_ID") VALUES (13,2,NULL);
INSERT INTO "sample_profile" ("SAMPLE_ID","GENETIC_PROFILE_ID","PANEL_ID") VALUES (13,3,NULL);
INSERT INTO "sample_profile" ("SAMPLE_ID","GENETIC_PROFILE_ID","PANEL_ID") VALUES (13,4,NULL);
INSERT INTO "sample_profile" ("SAMPLE_ID","GENETIC_PROFILE_ID","PANEL_ID") VALUES (14,2,NULL);
INSERT INTO "sample_profile" ("SAMPLE_ID","GENETIC_PROFILE_ID","PANEL_ID") VALUES (14,4,NULL);

-- sample_list
PRINT('LOADING TABLE sample_list')
SET IDENTITY_INSERT [sample_list] ON
INSERT INTO "sample_list" ("LIST_ID", "STABLE_ID", "CATEGORY", "CANCER_STUDY_ID", "NAME", "DESCRIPTION") VALUES (1,'study_tcga_pub_all','other',1,'All Tumors','All tumor samples');
INSERT INTO "sample_list" ("LIST_ID", "STABLE_ID", "CATEGORY", "CANCER_STUDY_ID", "NAME", "DESCRIPTION") VALUES (2,'study_tcga_pub_acgh','other',1,'Tumors aCGH','All tumors with aCGH data');
INSERT INTO "sample_list" ("LIST_ID", "STABLE_ID", "CATEGORY", "CANCER_STUDY_ID", "NAME", "DESCRIPTION") VALUES (3,'study_tcga_pub_cnaseq','other',1,'Tumors with sequencing and aCGH data','All tumor samples that have CNA and sequencing data');
INSERT INTO "sample_list" ("LIST_ID", "STABLE_ID", "CATEGORY", "CANCER_STUDY_ID", "NAME", "DESCRIPTION") VALUES (4,'study_tcga_pub_complete','other',1,'Complete samples (mutations, copy-number, expression)','Samples with complete data');
INSERT INTO "sample_list" ("LIST_ID", "STABLE_ID", "CATEGORY", "CANCER_STUDY_ID", "NAME", "DESCRIPTION") VALUES (5,'study_tcga_pub_log2CNA','other',1,'Tumors log2 copy-number','All tumors with log2 copy-number data');
INSERT INTO "sample_list" ("LIST_ID", "STABLE_ID", "CATEGORY", "CANCER_STUDY_ID", "NAME", "DESCRIPTION") VALUES (6,'study_tcga_pub_methylation_hm27','all_cases_with_mutation_data',1,'Tumors with methylation data','All samples with methylation (HM27) data');
INSERT INTO "sample_list" ("LIST_ID", "STABLE_ID", "CATEGORY", "CANCER_STUDY_ID", "NAME", "DESCRIPTION") VALUES (7,'study_tcga_pub_mrna','other',1,'Tumors with mRNA data (Agilent microarray)','All samples with mRNA expression data');
INSERT INTO "sample_list" ("LIST_ID", "STABLE_ID", "CATEGORY", "CANCER_STUDY_ID", "NAME", "DESCRIPTION") VALUES (8,'study_tcga_pub_sequenced','other',1,'Sequenced Tumors','All sequenced samples');
INSERT INTO "sample_list" ("LIST_ID", "STABLE_ID", "CATEGORY", "CANCER_STUDY_ID", "NAME", "DESCRIPTION") VALUES (9,'study_tcga_pub_cna','other',1,'Tumor Samples with CNA data','All tumors with CNA data');
INSERT INTO "sample_list" ("LIST_ID", "STABLE_ID", "CATEGORY", "CANCER_STUDY_ID", "NAME", "DESCRIPTION") VALUES (10,'study_tcga_pub_rna_seq_v2_mrna','other',1,'Tumor Samples with mRNA data (RNA Seq V2)','All samples with mRNA expression data');
INSERT INTO "sample_list" ("LIST_ID", "STABLE_ID", "CATEGORY", "CANCER_STUDY_ID", "NAME", "DESCRIPTION") VALUES (11,'study_tcga_pub_microrna','other',1,'Tumors with microRNA data (microRNA-Seq)','All samples with microRNA data');
INSERT INTO "sample_list" ("LIST_ID", "STABLE_ID", "CATEGORY", "CANCER_STUDY_ID", "NAME", "DESCRIPTION") VALUES (12,'study_tcga_pub_rppa','other',1,'Tumor Samples with RPPA data','Tumors with reverse phase protein array (RPPA) data for about 200 antibodies');
INSERT INTO "sample_list" ("LIST_ID", "STABLE_ID", "CATEGORY", "CANCER_STUDY_ID", "NAME", "DESCRIPTION") VALUES (13,'study_tcga_pub_3way_complete','other',1,'All Complete Tumors','All tumor samples that have mRNA, CNA and sequencing data');
INSERT INTO "sample_list" ("LIST_ID", "STABLE_ID", "CATEGORY", "CANCER_STUDY_ID", "NAME", "DESCRIPTION") VALUES (14,'acc_tcga_all','other',2,'All Tumors','All tumor samples');
SET IDENTITY_INSERT [sample_list] OFF

-- sample_list_list
PRINT('LOADING TABLE sample_list_list')
INSERT INTO "sample_list_list" ("LIST_ID","SAMPLE_ID") VALUES (1,1);
INSERT INTO "sample_list_list" ("LIST_ID","SAMPLE_ID") VALUES (1,2);
INSERT INTO "sample_list_list" ("LIST_ID","SAMPLE_ID") VALUES (1,3);
INSERT INTO "sample_list_list" ("LIST_ID","SAMPLE_ID") VALUES (1,4);
INSERT INTO "sample_list_list" ("LIST_ID","SAMPLE_ID") VALUES (1,5);
INSERT INTO "sample_list_list" ("LIST_ID","SAMPLE_ID") VALUES (1,6);
INSERT INTO "sample_list_list" ("LIST_ID","SAMPLE_ID") VALUES (1,7);
INSERT INTO "sample_list_list" ("LIST_ID","SAMPLE_ID") VALUES (1,8);
INSERT INTO "sample_list_list" ("LIST_ID","SAMPLE_ID") VALUES (1,9);
INSERT INTO "sample_list_list" ("LIST_ID","SAMPLE_ID") VALUES (1,10);
INSERT INTO "sample_list_list" ("LIST_ID","SAMPLE_ID") VALUES (1,11);
INSERT INTO "sample_list_list" ("LIST_ID","SAMPLE_ID") VALUES (1,12);
INSERT INTO "sample_list_list" ("LIST_ID","SAMPLE_ID") VALUES (1,13);
INSERT INTO "sample_list_list" ("LIST_ID","SAMPLE_ID") VALUES (1,14);
INSERT INTO "sample_list_list" ("LIST_ID","SAMPLE_ID") VALUES (2,1);
INSERT INTO "sample_list_list" ("LIST_ID","SAMPLE_ID") VALUES (2,2);
INSERT INTO "sample_list_list" ("LIST_ID","SAMPLE_ID") VALUES (2,3);
INSERT INTO "sample_list_list" ("LIST_ID","SAMPLE_ID") VALUES (2,4);
INSERT INTO "sample_list_list" ("LIST_ID","SAMPLE_ID") VALUES (2,5);
INSERT INTO "sample_list_list" ("LIST_ID","SAMPLE_ID") VALUES (2,6);
INSERT INTO "sample_list_list" ("LIST_ID","SAMPLE_ID") VALUES (2,7);
INSERT INTO "sample_list_list" ("LIST_ID","SAMPLE_ID") VALUES (2,8);
INSERT INTO "sample_list_list" ("LIST_ID","SAMPLE_ID") VALUES (2,9);
INSERT INTO "sample_list_list" ("LIST_ID","SAMPLE_ID") VALUES (2,10);
INSERT INTO "sample_list_list" ("LIST_ID","SAMPLE_ID") VALUES (2,11);
INSERT INTO "sample_list_list" ("LIST_ID","SAMPLE_ID") VALUES (2,12);
INSERT INTO "sample_list_list" ("LIST_ID","SAMPLE_ID") VALUES (2,13);
INSERT INTO "sample_list_list" ("LIST_ID","SAMPLE_ID") VALUES (2,14);
INSERT INTO "sample_list_list" ("LIST_ID","SAMPLE_ID") VALUES (3,2);
INSERT INTO "sample_list_list" ("LIST_ID","SAMPLE_ID") VALUES (3,3);
INSERT INTO "sample_list_list" ("LIST_ID","SAMPLE_ID") VALUES (3,6);
INSERT INTO "sample_list_list" ("LIST_ID","SAMPLE_ID") VALUES (3,8);
INSERT INTO "sample_list_list" ("LIST_ID","SAMPLE_ID") VALUES (3,9);
INSERT INTO "sample_list_list" ("LIST_ID","SAMPLE_ID") VALUES (3,10);
INSERT INTO "sample_list_list" ("LIST_ID","SAMPLE_ID") VALUES (3,12);
INSERT INTO "sample_list_list" ("LIST_ID","SAMPLE_ID") VALUES (4,2);
INSERT INTO "sample_list_list" ("LIST_ID","SAMPLE_ID") VALUES (4,3);
INSERT INTO "sample_list_list" ("LIST_ID","SAMPLE_ID") VALUES (4,6);
INSERT INTO "sample_list_list" ("LIST_ID","SAMPLE_ID") VALUES (4,8);
INSERT INTO "sample_list_list" ("LIST_ID","SAMPLE_ID") VALUES (4,9);
INSERT INTO "sample_list_list" ("LIST_ID","SAMPLE_ID") VALUES (4,10);
INSERT INTO "sample_list_list" ("LIST_ID","SAMPLE_ID") VALUES (4,12);
INSERT INTO "sample_list_list" ("LIST_ID","SAMPLE_ID") VALUES (5,1);
INSERT INTO "sample_list_list" ("LIST_ID","SAMPLE_ID") VALUES (5,2);
INSERT INTO "sample_list_list" ("LIST_ID","SAMPLE_ID") VALUES (5,3);
INSERT INTO "sample_list_list" ("LIST_ID","SAMPLE_ID") VALUES (5,4);
INSERT INTO "sample_list_list" ("LIST_ID","SAMPLE_ID") VALUES (5,5);
INSERT INTO "sample_list_list" ("LIST_ID","SAMPLE_ID") VALUES (5,6);
INSERT INTO "sample_list_list" ("LIST_ID","SAMPLE_ID") VALUES (5,7);
INSERT INTO "sample_list_list" ("LIST_ID","SAMPLE_ID") VALUES (5,8);
INSERT INTO "sample_list_list" ("LIST_ID","SAMPLE_ID") VALUES (5,9);
INSERT INTO "sample_list_list" ("LIST_ID","SAMPLE_ID") VALUES (5,10);
INSERT INTO "sample_list_list" ("LIST_ID","SAMPLE_ID") VALUES (5,11);
INSERT INTO "sample_list_list" ("LIST_ID","SAMPLE_ID") VALUES (5,12);
INSERT INTO "sample_list_list" ("LIST_ID","SAMPLE_ID") VALUES (5,13);
INSERT INTO "sample_list_list" ("LIST_ID","SAMPLE_ID") VALUES (5,14);
INSERT INTO "sample_list_list" ("LIST_ID","SAMPLE_ID") VALUES (6,2);
INSERT INTO "sample_list_list" ("LIST_ID","SAMPLE_ID") VALUES (7,2);
INSERT INTO "sample_list_list" ("LIST_ID","SAMPLE_ID") VALUES (7,3);
INSERT INTO "sample_list_list" ("LIST_ID","SAMPLE_ID") VALUES (7,6);
INSERT INTO "sample_list_list" ("LIST_ID","SAMPLE_ID") VALUES (7,8);
INSERT INTO "sample_list_list" ("LIST_ID","SAMPLE_ID") VALUES (7,9);
INSERT INTO "sample_list_list" ("LIST_ID","SAMPLE_ID") VALUES (7,10);
INSERT INTO "sample_list_list" ("LIST_ID","SAMPLE_ID") VALUES (7,12);
INSERT INTO "sample_list_list" ("LIST_ID","SAMPLE_ID") VALUES (7,13);
INSERT INTO "sample_list_list" ("LIST_ID","SAMPLE_ID") VALUES (8,2);
INSERT INTO "sample_list_list" ("LIST_ID","SAMPLE_ID") VALUES (8,3);
INSERT INTO "sample_list_list" ("LIST_ID","SAMPLE_ID") VALUES (8,6);
INSERT INTO "sample_list_list" ("LIST_ID","SAMPLE_ID") VALUES (8,8);
INSERT INTO "sample_list_list" ("LIST_ID","SAMPLE_ID") VALUES (8,9);
INSERT INTO "sample_list_list" ("LIST_ID","SAMPLE_ID") VALUES (8,10);
INSERT INTO "sample_list_list" ("LIST_ID","SAMPLE_ID") VALUES (8,12);
INSERT INTO "sample_list_list" ("LIST_ID","SAMPLE_ID") VALUES (9,2);
INSERT INTO "sample_list_list" ("LIST_ID","SAMPLE_ID") VALUES (9,3);
INSERT INTO "sample_list_list" ("LIST_ID","SAMPLE_ID") VALUES (9,6);
INSERT INTO "sample_list_list" ("LIST_ID","SAMPLE_ID") VALUES (9,8);
INSERT INTO "sample_list_list" ("LIST_ID","SAMPLE_ID") VALUES (9,9);
INSERT INTO "sample_list_list" ("LIST_ID","SAMPLE_ID") VALUES (9,10);
INSERT INTO "sample_list_list" ("LIST_ID","SAMPLE_ID") VALUES (9,12);
INSERT INTO "sample_list_list" ("LIST_ID","SAMPLE_ID") VALUES (10,2);
INSERT INTO "sample_list_list" ("LIST_ID","SAMPLE_ID") VALUES (10,3);
INSERT INTO "sample_list_list" ("LIST_ID","SAMPLE_ID") VALUES (10,6);
INSERT INTO "sample_list_list" ("LIST_ID","SAMPLE_ID") VALUES (10,8);
INSERT INTO "sample_list_list" ("LIST_ID","SAMPLE_ID") VALUES (10,9);
INSERT INTO "sample_list_list" ("LIST_ID","SAMPLE_ID") VALUES (10,10);
INSERT INTO "sample_list_list" ("LIST_ID","SAMPLE_ID") VALUES (10,12);
INSERT INTO "sample_list_list" ("LIST_ID","SAMPLE_ID") VALUES (13,2);
INSERT INTO "sample_list_list" ("LIST_ID","SAMPLE_ID") VALUES (13,3);
INSERT INTO "sample_list_list" ("LIST_ID","SAMPLE_ID") VALUES (13,6);
INSERT INTO "sample_list_list" ("LIST_ID","SAMPLE_ID") VALUES (13,8);
INSERT INTO "sample_list_list" ("LIST_ID","SAMPLE_ID") VALUES (13,9);
INSERT INTO "sample_list_list" ("LIST_ID","SAMPLE_ID") VALUES (13,10);
INSERT INTO "sample_list_list" ("LIST_ID","SAMPLE_ID") VALUES (13,12);
INSERT INTO "sample_list_list" ("LIST_ID","SAMPLE_ID") VALUES (14,15);


PRINT('LOADING TABLE copy_number_seg')
SET IDENTITY_INSERT [copy_number_seg] ON --//JK-UPDATED
INSERT INTO "copy_number_seg" ("SEG_ID","CANCER_STUDY_ID","SAMPLE_ID","CHR","START","END","NUM_PROBES","SEGMENT_MEAN") VALUES (50236594, 1, 1, '1', 224556, 180057677, 291, 0.0519);
INSERT INTO "copy_number_seg" ("SEG_ID","CANCER_STUDY_ID","SAMPLE_ID","CHR","START","END","NUM_PROBES","SEGMENT_MEAN") VALUES (50236593, 1, 2,	'2', 1402650, 190262486, 207, 0.0265);
INSERT INTO "copy_number_seg" ("SEG_ID","CANCER_STUDY_ID","SAMPLE_ID","CHR","START","END","NUM_PROBES","SEGMENT_MEAN") VALUES (50236592, 1, 3,	'3', 1449872, 194238390, 341, 0.0347);
SET IDENTITY_INSERT [copy_number_seg] OFF


PRINT('LOADING TABLE clinical_patient')
INSERT INTO "clinical_patient" ("INTERNAL_ID", "ATTR_ID", "ATTR_VALUE") VALUES (1, 'RETROSPECTIVE_COLLECTION', 'NO');
INSERT INTO "clinical_patient" ("INTERNAL_ID", "ATTR_ID", "ATTR_VALUE") VALUES (1, 'FORM_COMPLETION_DATE', '2013-12-5');
INSERT INTO "clinical_patient" ("INTERNAL_ID", "ATTR_ID", "ATTR_VALUE") VALUES (1, 'OTHER_PATIENT_ID', '286CF147-B7F7-4A05-8E41-7FBD3717AD71');
INSERT INTO "clinical_patient" ("INTERNAL_ID", "ATTR_ID", "ATTR_VALUE") VALUES (2, 'PROSPECTIVE_COLLECTION', 'YES');
INSERT INTO "clinical_patient" ("INTERNAL_ID", "ATTR_ID", "ATTR_VALUE") VALUES (15, 'DFS_MONTHS', '5.72');
INSERT INTO "clinical_patient" ("INTERNAL_ID", "ATTR_ID", "ATTR_VALUE") VALUES (15, 'DFS_STATUS', 'Recurred/Progressed');
INSERT INTO "clinical_patient" ("INTERNAL_ID", "ATTR_ID", "ATTR_VALUE") VALUES (15, 'OS_MONTHS', 'Recurred/Progressed');
INSERT INTO "clinical_patient" ("INTERNAL_ID", "ATTR_ID", "ATTR_VALUE") VALUES (15, 'OS_STATUS', 'LIVING');

PRINT('LOADING TABLE clinical_sample')
INSERT INTO "clinical_sample" ("INTERNAL_ID", "ATTR_ID", "ATTR_VALUE") VALUES (1, 'OTHER_SAMPLE_ID', '5C631CE8-F96A-4C35-A459-556FC4AB21E1');
INSERT INTO "clinical_sample" ("INTERNAL_ID", "ATTR_ID", "ATTR_VALUE") VALUES (1, 'DAYS_TO_COLLECTION', '276');
INSERT INTO "clinical_sample" ("INTERNAL_ID", "ATTR_ID", "ATTR_VALUE") VALUES (1, 'IS_FFPE', 'NO');
INSERT INTO "clinical_sample" ("INTERNAL_ID", "ATTR_ID", "ATTR_VALUE") VALUES (2, 'OCT_EMBEDDED', 'false');
INSERT INTO "clinical_sample" ("INTERNAL_ID", "ATTR_ID", "ATTR_VALUE") VALUES (2, 'PATHOLOGY_REPORT_FILE_NAME', 'TCGA-GC-A3BM.F3408556-9259-4700-B9A0-F41E516B420C.pdf');
INSERT INTO "clinical_sample" ("INTERNAL_ID", "ATTR_ID", "ATTR_VALUE") VALUES (2, 'SAMPLE_TYPE', 'Primary Tumor');
INSERT INTO "clinical_sample" ("INTERNAL_ID", "ATTR_ID", "ATTR_VALUE") VALUES (15, 'OTHER_SAMPLE_ID', '91E7F41C-17B3-4724-96EF-D3C207B964E1');
INSERT INTO "clinical_sample" ("INTERNAL_ID", "ATTR_ID", "ATTR_VALUE") VALUES (15, 'DAYS_TO_COLLECTION', '111');

PRINT('LOADING TABLE clinical_attribute_meta')
INSERT INTO "clinical_attribute_meta" ("ATTR_ID", "DISPLAY_NAME", "DESCRIPTION", "DATATYPE", "PATIENT_ATTRIBUTE", "PRIORITY", "CANCER_STUDY_ID") VALUES ('RETROSPECTIVE_COLLECTION', 'Tissue Retrospective Collection Indicator', 'Text indicator for the time frame of tissue procurement, indicating that the tissue was obtained and stored prior to the initiation of the project.', 'STRING', 1, '1', 1);
INSERT INTO "clinical_attribute_meta" ("ATTR_ID", "DISPLAY_NAME", "DESCRIPTION", "DATATYPE", "PATIENT_ATTRIBUTE", "PRIORITY", "CANCER_STUDY_ID") VALUES ('PROSPECTIVE_COLLECTION', 'Tissue Prospective Collection Indicator', 'Text indicator for the time frame of tissue procurement, indicating that the tissue was procured in parallel to the project.', 'STRING', 1, '1', 1);
INSERT INTO "clinical_attribute_meta" ("ATTR_ID", "DISPLAY_NAME", "DESCRIPTION", "DATATYPE", "PATIENT_ATTRIBUTE", "PRIORITY", "CANCER_STUDY_ID") VALUES ('FORM_COMPLETION_DATE', 'Form completion date', 'Form completion date', 'STRING', 1, '1', 1);
INSERT INTO "clinical_attribute_meta" ("ATTR_ID", "DISPLAY_NAME", "DESCRIPTION", "DATATYPE", "PATIENT_ATTRIBUTE", "PRIORITY", "CANCER_STUDY_ID") VALUES ('OTHER_PATIENT_ID', 'Other Patient ID', 'Legacy DMP patient identifier (DMPnnnn)', 'STRING', 1, '1', 1);
INSERT INTO "clinical_attribute_meta" ("ATTR_ID", "DISPLAY_NAME", "DESCRIPTION", "DATATYPE", "PATIENT_ATTRIBUTE", "PRIORITY", "CANCER_STUDY_ID") VALUES ('DFS_MONTHS', 'Disease Free (Months)', 'Disease free (months) since initial treatment.', 'NUMBER', 1, '1', 1);
INSERT INTO "clinical_attribute_meta" ("ATTR_ID", "DISPLAY_NAME", "DESCRIPTION", "DATATYPE", "PATIENT_ATTRIBUTE", "PRIORITY", "CANCER_STUDY_ID") VALUES ('DFS_STATUS', 'Disease Free Status', 'Disease free status since initial treatment.', 'STRING', 1, '1', 1);
INSERT INTO "clinical_attribute_meta" ("ATTR_ID", "DISPLAY_NAME", "DESCRIPTION", "DATATYPE", "PATIENT_ATTRIBUTE", "PRIORITY", "CANCER_STUDY_ID") VALUES ('OS_MONTHS', 'Overall Survival (Months)', 'Overall survival in months since initial diagonosis.', 'NUMBER', 1, '1', 1);
INSERT INTO "clinical_attribute_meta" ("ATTR_ID", "DISPLAY_NAME", "DESCRIPTION", "DATATYPE", "PATIENT_ATTRIBUTE", "PRIORITY", "CANCER_STUDY_ID") VALUES ('OS_STATUS', 'Overall Survival Status', 'Overall patient survival status.', 'STRING', 1, '1', 1);
INSERT INTO "clinical_attribute_meta" ("ATTR_ID", "DISPLAY_NAME", "DESCRIPTION", "DATATYPE", "PATIENT_ATTRIBUTE", "PRIORITY", "CANCER_STUDY_ID") VALUES ('OTHER_SAMPLE_ID', 'Other Sample ID', 'Legacy DMP sample identifier (DMPnnnn)', 'STRING', 0, '1', 1);
INSERT INTO "clinical_attribute_meta" ("ATTR_ID", "DISPLAY_NAME", "DESCRIPTION", "DATATYPE", "PATIENT_ATTRIBUTE", "PRIORITY", "CANCER_STUDY_ID") VALUES ('DAYS_TO_COLLECTION', 'Days to Sample Collection.', 'Days to sample collection.', 'STRING', 0, '1', 1);
INSERT INTO "clinical_attribute_meta" ("ATTR_ID", "DISPLAY_NAME", "DESCRIPTION", "DATATYPE", "PATIENT_ATTRIBUTE", "PRIORITY", "CANCER_STUDY_ID") VALUES ('IS_FFPE', 'Is FFPE', 'If the sample is from FFPE', 'STRING', 0, '1', 1);
INSERT INTO "clinical_attribute_meta" ("ATTR_ID", "DISPLAY_NAME", "DESCRIPTION", "DATATYPE", "PATIENT_ATTRIBUTE", "PRIORITY", "CANCER_STUDY_ID") VALUES ('OCT_EMBEDDED', 'Oct embedded', 'Oct embedded', 'STRING', 0, '1', 1);
INSERT INTO "clinical_attribute_meta" ("ATTR_ID", "DISPLAY_NAME", "DESCRIPTION", "DATATYPE", "PATIENT_ATTRIBUTE", "PRIORITY", "CANCER_STUDY_ID") VALUES ('PATHOLOGY_REPORT_FILE_NAME', 'Pathology Report File Name', 'Pathology Report File Name', 'STRING', 0, '1', 1);
INSERT INTO "clinical_attribute_meta" ("ATTR_ID", "DISPLAY_NAME", "DESCRIPTION", "DATATYPE", "PATIENT_ATTRIBUTE", "PRIORITY", "CANCER_STUDY_ID") VALUES ('SAMPLE_TYPE', 'Sample Type', 'The type of sample (i.e., normal, primary, met, recurrence).', 'STRING', 0, '1', 1);
INSERT INTO "clinical_attribute_meta" ("ATTR_ID", "DISPLAY_NAME", "DESCRIPTION", "DATATYPE", "PATIENT_ATTRIBUTE", "PRIORITY", "CANCER_STUDY_ID") VALUES ('RETROSPECTIVE_COLLECTION', 'Tissue Retrospective Collection Indicator', 'Text indicator for the time frame of tissue procurement, indicating that the tissue was obtained and stored prior to the initiation of the project.', 'STRING', 1, '1', 2);
INSERT INTO "clinical_attribute_meta" ("ATTR_ID", "DISPLAY_NAME", "DESCRIPTION", "DATATYPE", "PATIENT_ATTRIBUTE", "PRIORITY", "CANCER_STUDY_ID") VALUES ('PROSPECTIVE_COLLECTION', 'Tissue Prospective Collection Indicator', 'Text indicator for the time frame of tissue procurement, indicating that the tissue was procured in parallel to the project.', 'STRING', 1, '1', 2);
INSERT INTO "clinical_attribute_meta" ("ATTR_ID", "DISPLAY_NAME", "DESCRIPTION", "DATATYPE", "PATIENT_ATTRIBUTE", "PRIORITY", "CANCER_STUDY_ID") VALUES ('FORM_COMPLETION_DATE', 'Form completion date', 'Form completion date', 'STRING', 1, '1', 2);
INSERT INTO "clinical_attribute_meta" ("ATTR_ID", "DISPLAY_NAME", "DESCRIPTION", "DATATYPE", "PATIENT_ATTRIBUTE", "PRIORITY", "CANCER_STUDY_ID") VALUES ('OTHER_PATIENT_ID', 'Other Patient ID', 'Legacy DMP patient identifier (DMPnnnn)', 'STRING', 1, '1', 2);
INSERT INTO "clinical_attribute_meta" ("ATTR_ID", "DISPLAY_NAME", "DESCRIPTION", "DATATYPE", "PATIENT_ATTRIBUTE", "PRIORITY", "CANCER_STUDY_ID") VALUES ('DFS_MONTHS', 'Disease Free (Months)', 'Disease free (months) since initial treatment.', 'NUMBER', 1, '1', 2);
INSERT INTO "clinical_attribute_meta" ("ATTR_ID", "DISPLAY_NAME", "DESCRIPTION", "DATATYPE", "PATIENT_ATTRIBUTE", "PRIORITY", "CANCER_STUDY_ID") VALUES ('DFS_STATUS', 'Disease Free Status', 'Disease free status since initial treatment.', 'STRING', 1, '1', 2);
INSERT INTO "clinical_attribute_meta" ("ATTR_ID", "DISPLAY_NAME", "DESCRIPTION", "DATATYPE", "PATIENT_ATTRIBUTE", "PRIORITY", "CANCER_STUDY_ID") VALUES ('OS_MONTHS', 'Overall Survival (Months)', 'Overall survival in months since initial diagonosis.', 'NUMBER', 1, '1', 2);
INSERT INTO "clinical_attribute_meta" ("ATTR_ID", "DISPLAY_NAME", "DESCRIPTION", "DATATYPE", "PATIENT_ATTRIBUTE", "PRIORITY", "CANCER_STUDY_ID") VALUES ('OS_STATUS', 'Overall Survival Status', 'Overall patient survival status.', 'STRING', 1, '1', 2);
INSERT INTO "clinical_attribute_meta" ("ATTR_ID", "DISPLAY_NAME", "DESCRIPTION", "DATATYPE", "PATIENT_ATTRIBUTE", "PRIORITY", "CANCER_STUDY_ID") VALUES ('OTHER_SAMPLE_ID', 'Other Sample ID', 'Legacy DMP sample identifier (DMPnnnn)', 'STRING', 0, '1', 2);
INSERT INTO "clinical_attribute_meta" ("ATTR_ID", "DISPLAY_NAME", "DESCRIPTION", "DATATYPE", "PATIENT_ATTRIBUTE", "PRIORITY", "CANCER_STUDY_ID") VALUES ('DAYS_TO_COLLECTION', 'Days to Sample Collection.', 'Days to sample collection.', 'STRING', 0, '1', 2);
INSERT INTO "clinical_attribute_meta" ("ATTR_ID", "DISPLAY_NAME", "DESCRIPTION", "DATATYPE", "PATIENT_ATTRIBUTE", "PRIORITY", "CANCER_STUDY_ID") VALUES ('IS_FFPE', 'Is FFPE', 'If the sample is from FFPE', 'STRING', 0, '1', 2);
INSERT INTO "clinical_attribute_meta" ("ATTR_ID", "DISPLAY_NAME", "DESCRIPTION", "DATATYPE", "PATIENT_ATTRIBUTE", "PRIORITY", "CANCER_STUDY_ID") VALUES ('OCT_EMBEDDED', 'Oct embedded', 'Oct embedded', 'STRING', 0, '1', 2);
INSERT INTO "clinical_attribute_meta" ("ATTR_ID", "DISPLAY_NAME", "DESCRIPTION", "DATATYPE", "PATIENT_ATTRIBUTE", "PRIORITY", "CANCER_STUDY_ID") VALUES ('PATHOLOGY_REPORT_FILE_NAME', 'Pathology Report File Name', 'Pathology Report File Name', 'STRING', 0, '1', 2);
INSERT INTO "clinical_attribute_meta" ("ATTR_ID", "DISPLAY_NAME", "DESCRIPTION", "DATATYPE", "PATIENT_ATTRIBUTE", "PRIORITY", "CANCER_STUDY_ID") VALUES ('SAMPLE_TYPE', 'Sample Type', 'The type of sample (i.e., normal, primary, met, recurrence).', 'STRING', 0, '1', 2);

-- structural_variant
PRINT('LOADING TABLE structural_variant')
INSERT INTO "structural_variant" ("SAMPLE_ID","BREAKPOINT_TYPE","ANNOTATION","COMMENTS","CONFIDENCE_CLASS","CONNECTION_TYPE","EVENT_INFO","MAPQ","NORMAL_READ_COUNT","NORMAL_VARIANT_COUNT","PAIRED_END_READ_SUPPORT","SITE1_CHROM","SITE1_DESC","SITE1_ENTREZ_GENE_ID","SITE1_POS","SITE2_CHROM","SITE2_DESC","SITE2_ENTREZ_GENE_ID","SITE2_POS","SPLIT_READ_SUPPORT","SV_CLASS_NAME","SV_DESC","SV_LENGTH","TUMOR_READ_COUNT","TUMOR_VARIANT_COUNT","VARIANT_STATUS_NAME","GENETIC_PROFILE_ID") VALUES (1,'PRECISE','n/a','n/a','AUTO_OK','3to5','Transcript fusion (ERBB2-GRB7)',0,7181,0,11,'17','Intron of ERBB2(+): 51bp after exon 26',2064,37883851,'17','5-UTR of GRB7(+): 1Kb before coding start',2886,37897379,31,'DELETION','n/a',13528,4389,6,'NEW_VARIANT',7);
INSERT INTO "structural_variant" ("SAMPLE_ID","BREAKPOINT_TYPE","ANNOTATION","COMMENTS","CONFIDENCE_CLASS","CONNECTION_TYPE","EVENT_INFO","MAPQ","NORMAL_READ_COUNT","NORMAL_VARIANT_COUNT","PAIRED_END_READ_SUPPORT","SITE1_CHROM","SITE1_DESC","SITE1_ENTREZ_GENE_ID","SITE1_POS","SITE2_CHROM","SITE2_DESC","SITE2_ENTREZ_GENE_ID","SITE2_POS","SPLIT_READ_SUPPORT","SV_CLASS_NAME","SV_DESC","SV_LENGTH","TUMOR_READ_COUNT","TUMOR_VARIANT_COUNT","VARIANT_STATUS_NAME","GENETIC_PROFILE_ID") VALUES (2,'PRECISE','n/a','n/a','AUTO_OK','3to5','Protein fusion: mid-exon (ERBB2-GRB7)',0,7062,0,97,'17','Exon 25 of ERBB2(+)',2064,37883138,'17','Intron of GRB7(+): 56bp before exon 10',2886,37901416,71,'DELETION','n/a',18278,9849,60,'NEW_VARIANT',7);
INSERT INTO "structural_variant" ("SAMPLE_ID","BREAKPOINT_TYPE","ANNOTATION","COMMENTS","CONFIDENCE_CLASS","CONNECTION_TYPE","EVENT_INFO","MAPQ","NORMAL_READ_COUNT","NORMAL_VARIANT_COUNT","PAIRED_END_READ_SUPPORT","SITE1_CHROM","SITE1_DESC","SITE1_ENTREZ_GENE_ID","SITE1_POS","SITE2_CHROM","SITE2_DESC","SITE2_ENTREZ_GENE_ID","SITE2_POS","SPLIT_READ_SUPPORT","SV_CLASS_NAME","SV_DESC","SV_LENGTH","TUMOR_READ_COUNT","TUMOR_VARIANT_COUNT","VARIANT_STATUS_NAME","GENETIC_PROFILE_ID") VALUES (3,'PRECISE','n/a','ERBB2 (NM_004448) rearrangement','MANUAL_OK','3to5','Deletion of 1 exon: in frame',0,7212,0,31,'17','Intron of ERBB2(+): 46bp after exon 15',2064,37873779,'17','Intron of ERBB2(+): 501bp before exon 17',2064,37879041,10,'DELETION','n/a',5262,3101,7,'NEW_VARIANT',7);
INSERT INTO "structural_variant" ("SAMPLE_ID","BREAKPOINT_TYPE","ANNOTATION","COMMENTS","CONFIDENCE_CLASS","CONNECTION_TYPE","EVENT_INFO","MAPQ","NORMAL_READ_COUNT","NORMAL_VARIANT_COUNT","PAIRED_END_READ_SUPPORT","SITE1_CHROM","SITE1_DESC","SITE1_ENTREZ_GENE_ID","SITE1_POS","SITE2_CHROM","SITE2_DESC","SITE2_ENTREZ_GENE_ID","SITE2_POS","SPLIT_READ_SUPPORT","SV_CLASS_NAME","SV_DESC","SV_LENGTH","TUMOR_READ_COUNT","TUMOR_VARIANT_COUNT","VARIANT_STATUS_NAME","GENETIC_PROFILE_ID") VALUES (4,'PRECISE','n/a','n/a','MANUAL_OK','3to5','Deletion of 1 exon',0,5328,0,52,'17','Intron of ERBB2(+): 196bp after exon 15',2064,37873929,'17','Intron of ERBB2(+): 442bp after exon 16',2064,37876529,33,'DELETION','n/a',2600,19320,13,'NEW_VARIANT',7);

